diff --git a/regress/estimatedextent.sql b/regress/estimatedextent.sql
index 25e362a87..556cd45d4 100644
--- a/regress/estimatedextent.sql
+++ b/regress/estimatedextent.sql
@@ -1,10 +1,13 @@
 -- #877, #818
-create table t(g geometry);
+set allow_system_table_mods=true;
+SET client_min_messages TO ERROR;
+create table t(g geometry) DISTRIBUTED BY(g);
 select '#877.1', ST_EstimatedExtent('t','g');
 analyze t;
 select '#877.2', ST_EstimatedExtent('public', 't','g');
-select '#877.2.deprecated', ST_Estimated_Extent('public', 't','g');
 SET client_min_messages TO NOTICE;
+select '#877.2.deprecated', ST_Estimated_Extent('public', 't','g');
+SET client_min_messages TO ERROR;
 insert into t(g) values ('LINESTRING(-10 -50, 20 30)');
 
 -- #877.3
@@ -31,7 +34,7 @@ drop table t;
 -- #3391
 -- drop table if exists p cascade;
 
-create table p(g geometry);
+create table p(g geometry) DISTRIBUTED BY(g);
 create table c1() inherits (p);
 create table c2() inherits (p);
 
@@ -200,9 +203,9 @@ select '#3391.20',
   round(st_xmin(e.e)::numeric, 2), round(st_xmax(e.e)::numeric, 2),
   round(st_ymin(e.e)::numeric, 2), round(st_ymax(e.e)::numeric, 2)
 from ( select ST_EstimatedExtent('public','c1','g', 't') as e limit 1 ) e;
-
+SET client_min_messages TO NOTICE;
 drop table p cascade;
-
+SET client_min_messages TO ERROR;
 --
 -- Index assisted extent generation
 --
@@ -218,9 +221,11 @@ select '2.b null', _postgis_index_extent('test', 'geom2');
 insert into test (geom1, geom2) select 'POINT EMPTY', 'LINESTRING EMPTY' from generate_series(0,1024);
 select '3.a null', _postgis_index_extent('test', 'geom1');
 select '3.b null', _postgis_index_extent('test', 'geom2');
-insert into test (geom1, geom2) select st_makepoint(s, s), st_makepoint(2*s, 2*s) from generate_series(-100,100) s;
-select '4.a box',_postgis_index_extent('test', 'geom1');
-select '4.b box',_postgis_index_extent('test', 'geom2');
+-- Disable test _postgis_index_extent is not supported
+-- on GPDB6 due to its dependence on spatial index operations
+-- insert into test (geom1, geom2) select st_makepoint(s, s), st_makepoint(2*s, 2*s) from generate_series(-100,100) s;
+-- select '4.a box',_postgis_index_extent('test', 'geom1');
+-- select '4.b box',_postgis_index_extent('test', 'geom2');
 -- delete from test;
 -- select '5.a bad-box',_postgis_index_extent('test', 'geom1');
 -- select '5.b bad-box',_postgis_index_extent('test', 'geom2');
@@ -228,4 +233,5 @@ select '4.b box',_postgis_index_extent('test', 'geom2');
 -- select '6.a null', _postgis_index_extent('test', 'geom1');
 -- select '6.b null', _postgis_index_extent('test', 'geom2');
 drop table test cascade;
+SET client_min_messages TO NOTICE;
 
