diff --git a/regress/in_gml_expected b/regress/in_gml_expected
index 6fa4e2d6d..119d79268 100644
--- a/regress/in_gml_expected
+++ b/regress/in_gml_expected
@@ -1,94 +1,94 @@
 empty_geom|
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 point_1|POINT(1 2)
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 point_3|POINT EMPTY
 point_4|SRID=4326;POINT(1 2)
 linestring_1|LINESTRING(1 2,3 4)
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 linestring_3|SRID=4326;LINESTRING(1 2,3 4)
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 linestring_5|LINESTRING EMPTY
 linestring_6|LINESTRING(1 2,3 4)
 curve_1|LINESTRING(1 2,3 4)
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 curve_3|SRID=4326;LINESTRING(1 2,3 4)
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 curve_8|LINESTRING(1 2,3 4)
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 curve_10|LINESTRING(1 2,3 4)
 curve_11|LINESTRING(1 2,3 4,5 6)
 curve_12|LINESTRING(1 2 3,4 5 6,7 8 9)
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 curve_15|LINESTRING(1 2,4 5,7 8)
 curve_16|LINESTRING(1 2,3 4,6 7)
 polygon_1|POLYGON((1 2,3 4,5 6,1 2))
 polygon_2|SRID=4326;POLYGON((1 2,3 4,5 6,1 2))
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 polygon_9|POLYGON EMPTY
 polygon_10|POLYGON((1 2,3 4,5 6,1 2),(7 8,9 10,11 12,7 8))
 polygon_11|POLYGON((1 2,3 4,5 6,1 2),(7 8,9 10,11 12,7 8))
 polygon_12|POLYGON((1 2,3 4,5 6,1 2))
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 polygon_16|POLYGON((1 2,3 4,5 6,1 2),(7 8,9 10,11 12,7 8),(13 14,15 16,17 18,13 14))
 polygon_17|POLYGON((1 2,3 4,5 6,1 2),(7 8,9 10,11 12,7 8))
 polygon_18|POLYGON((1 2,4 5,7 8,1 2),(10 11,12 13,14 15,10 11))
 polygon_19|POLYGON((1 2,3 4,5 6,1 2),(7 8,10 11,13 14,7 8))
 linearring_1|POLYGON((1 2,3 4,5 6,1 2))
 linearring_2|SRID=4326;POLYGON((1 2,3 4,5 6,1 2))
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 linearring_8|POLYGON((1 2,3 4,5 6,1 2))
 triangle_1|TRIANGLE((1 2,3 4,5 6,1 2))
 triangle_2|SRID=4326;TRIANGLE((1 2,3 4,5 6,1 2))
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 triangle_8|TRIANGLE EMPTY
 triangle_9|TRIANGLE((1 2,3 4,5 6,1 2))
 triangle_10|TRIANGLE((1 2,3 4,5 6,1 2))
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 surface_1|POLYGON((1 2,3 4,5 6,1 2))
 surface_2|SRID=4326;POLYGON((1 2,3 4,5 6,1 2))
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 surface_11|POLYGON((1 2,3 4,5 6,1 2),(7 8,9 10,11 12,7 8))
 surface_12|POLYGON((1 2,3 4,5 6,1 2),(7 8,9 10,11 12,7 8))
 surface_13|POLYGON((1 2,3 4,5 6,1 2))
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 surface_17|POLYGON((1 2,3 4,5 6,1 2),(7 8,9 10,11 12,7 8),(13 14,15 16,17 18,13 14))
 surface_18|POLYGON((1 2,4 5,7 8,1 2),(10 11,12 13,14 15,10 11))
 surface_19|POLYGON((1 2,3 4,5 6,1 2),(7 8,10 11,13 14,7 8))
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 surface_21|POLYGON((1 2,3 4,5 6,1 2))
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 mpoint_1|MULTIPOINT(1 2)
 mpoint_2|MULTIPOINT(1 2,3 4)
 mpoint_3|SRID=4326;MULTIPOINT(1 2)
@@ -143,12 +143,12 @@ msurface_11|MULTIPOLYGON(((1 2,3 4,5 6,1 2)),((7 8,9 10,11 12,7 8)))
 msurface_12|MULTIPOLYGON EMPTY
 polyhedralsurface_1|POLYHEDRALSURFACE(((1 2 3,4 5 6,7 8 9,1 2 3)))
 polyhedralsurface_2|SRID=4326;POLYHEDRALSURFACE(((1 2 3,4 5 6,7 8 9,1 2 3)))
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 polyhedralsurface_9|POLYHEDRALSURFACE EMPTY
 polyhedralsurface_10|POLYHEDRALSURFACE EMPTY
 polyhedralsurface_11|POLYHEDRALSURFACE(((1 2 3,4 5 6,7 8 9,1 2 3),(10 11 12,13 14 15,16 17 18,10 11 12)))
@@ -156,23 +156,23 @@ polyhedralsurface_12|POLYHEDRALSURFACE(((1 2 3,4 5 6,7 8 9,1 2 3)),((10 11 12,13
 polyhedralsurface_13|POLYHEDRALSURFACE(((1 2 3,4 5 6,7 8 9,1 2 3)),((10 11 12,13 14 15,16 17 18,10 11 12),(19 20 21,22 23 24,25 26 27,19 20 21)))
 polyhedralsurface_14|POLYHEDRALSURFACE(((1 2 3,4 5 6,7 8 9,1 2 3)),((10 11 12,13 14 15,16 17 18,10 11 12),(19 20 21,22 23 24,25 26 27,19 20 21)))
 polyhedralsurface_15|POLYHEDRALSURFACE(((1 2 3,4 5 6,7 8 9,1 2 3)))
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 polyhedralsurface_19|POLYHEDRALSURFACE(((1 2 3,4 5 6,7 8 9,1 2 3),(10 11 12,13 14 15,16 17 18,10 11 12),(19 20 21,22 23 24,25 26 27,19 20 21)))
 polyhedralsurface_20|POLYHEDRALSURFACE(((1 2,4 5,7 8,1 2),(10 11,12 13,14 15,10 11)))
 polyhedralsurface_21|POLYHEDRALSURFACE(((1 2,3 4,5 6,1 2),(7 8,10 11,13 14,7 8)))
 polyhedralsurface_22|POLYHEDRALSURFACE(((1 2,4 5,7 8,1 2)),((10 11,12 13,14 15,10 11)))
 polyhedralsurface_23|POLYHEDRALSURFACE(((1 2,3 4,5 6,1 2)),((7 8,10 11,13 14,7 8)))
 polyhedralsurface_24|POLYHEDRALSURFACE(((1 2 3,4 5 6,7 8 9,1 2 3)))
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 tin_1|TIN(((1 2 3,4 5 6,7 8 9,1 2 3)))
 tin_2|SRID=4326;TIN(((1 2 3,4 5 6,7 8 9,1 2 3)))
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 tin_8|TIN EMPTY
 tin_9|TIN EMPTY
 tin_10|TIN EMPTY
@@ -206,19 +206,19 @@ srs_5|SRID=4326;POINT(2 1)
 srs_6|SRID=4326;POINT(2 1)
 srs_7|SRID=4326;POINT(1 2)
 srs_8|SRID=4326;POINT(1 2)
-ERROR:  unknown spatial reference system
-ERROR:  unknown spatial reference system
-ERROR:  unknown spatial reference system
-ERROR:  unknown spatial reference system
-ERROR:  unknown spatial reference system
-ERROR:  unknown spatial reference system
-ERROR:  unknown spatial reference system
-ERROR:  unknown spatial reference system
+ERROR:  unknown spatial reference system (lwgeom_pg.c:128)
+ERROR:  unknown spatial reference system (lwgeom_pg.c:128)
+ERROR:  unknown spatial reference system (lwgeom_pg.c:128)
+ERROR:  unknown spatial reference system (lwgeom_pg.c:128)
+ERROR:  unknown spatial reference system (lwgeom_pg.c:128)
+ERROR:  unknown spatial reference system (lwgeom_pg.c:128)
+ERROR:  unknown spatial reference system (lwgeom_pg.c:128)
+ERROR:  unknown spatial reference system (lwgeom_pg.c:128)
 srs_17|SRID=4326;GEOMETRYCOLLECTION(POINT(2 1),LINESTRING(4 3,6 5),LINESTRING(8 7,10 9),POLYGON((12 11,14 13,16 15,12 11),(18 17,20 19,22 21,18 17)),POLYGON((24 23,26 25,28 27,24 23),(26 25,28 27,30 29,26 25)))
 ns_1|POINT(1 2)
 ns_2|POINT(1 2)
 ns_3|POINT(1 2)
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 ns_5|POINT(1 2)
 ns_6|POINT(1 2)
 ns_7|SRID=4326;POINT(1 2)
@@ -226,91 +226,91 @@ ns_8|SRID=4326;POINT(1 2)
 ns_9|SRID=4326;POINT(1 2)
 ns_10|SRID=4326;POINT(1 2)
 ns_11|POINT(1 2)
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 ns_13|SRID=4326;POINT(1 2)
 coordinates_1|POINT(1 2)
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 coordinates_3|POINT(1 2 3)
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 coordinates_12|LINESTRING(1 2,3 4)
 coordinates_13|LINESTRING(1 2,3 4)
 coordinates_14|LINESTRING(1 2,4 5)
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 coordinates_cs_1|POINT(1 2)
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 coordinates_cs_3|POINT(1 2)
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 coordinates_cs_6|LINESTRING(1 2,3 4)
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 coordinates_cs_8|LINESTRING(1 2,3 4)
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 coordinates_cs_11|LINESTRING(1.1 2.2,3.3 4.4)
 coordinates_cs_12|LINESTRING(1.1 2.2,3.3 4.4)
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 coordinates_cs_15|LINESTRING(1.1 2.2,3.3 4.4)
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 coordinates_cs_19|LINESTRING(1 2,3 4)
 pos_1|POINT(1 2)
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 pos_8|POINT(1 2)
 pos_9|POINT(1 2)
 pos_10|POINT(1 2)
 pos_11|POINT(1 2)
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 pos_14|POINT(1 2)
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 pos_16|POINT(1 2 3)
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 poslist_1|LINESTRING(1 2,3 4)
 poslist_2|LINESTRING(1 2,3 4)
 poslist_3|LINESTRING(1 2,3 4)
 poslist_4|LINESTRING(1 2,3 4)
 poslist_5|LINESTRING(1 2,3 4)
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 poslist_8|LINESTRING(1 2 3,4 5 6)
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 poslist_16|LINESTRING(1 2,3 4)
 poslist_17|LINESTRING(1 2,3 4)
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 data_1|LINESTRING(1 2,3 4,5 6,7 8,9 10,11 12)
 data_2|LINESTRING(1 2,3 4,5 6,7 8,9 10)
 xlink_1|LINESTRING(1 2,1 2,3 4)
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 xlink_11|LINESTRING(1 2,1 2,3 4)
 xlink_12|GEOMETRYCOLLECTION(POINT(1 2),POINT(1 2))
 xlink_13|GEOMETRYCOLLECTION(LINESTRING(1 2,3 4),LINESTRING(1 2,3 4))
@@ -323,7 +323,7 @@ xlink_19|GEOMETRYCOLLECTION(MULTILINESTRING((1 2,3 4)),MULTILINESTRING((1 2,3 4)
 xlink_20|GEOMETRYCOLLECTION(MULTIPOLYGON(((1 2,3 4,5 6,1 2))),MULTIPOLYGON(((1 2,3 4,5 6,1 2))))
 xlink_21|GEOMETRYCOLLECTION(MULTIPOLYGON(((1 2,3 4,5 6,1 2))),MULTIPOLYGON(((1 2,3 4,5 6,1 2))))
 xlink_22|GEOMETRYCOLLECTION(GEOMETRYCOLLECTION(POINT(1 2)),GEOMETRYCOLLECTION(POINT(1 2)))
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 gml_1|POINT(1 2)
 gml_2|POINT(1 2 3)
 gml_3|SRID=27582;POINT(1 2)
@@ -387,13 +387,13 @@ gml_60|GEOMETRYCOLLECTION(POINT(1 2 3))
 gml_61|SRID=4326;GEOMETRYCOLLECTION(POINT(1 2 3))
 coord_1|POINT(1 2)
 coord_2|POINT(1 2 3)
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 coord_10|POINT(1 2)
 coord_11|POINT(1 2)
 coord_12|POINT(1 2)
@@ -404,17 +404,17 @@ coord_16|LINESTRING(1 2,3 4)
 double_1|POINT(1 1234567890)
 double_2|POINT(1 -1)
 double_3|POINT(1 1)
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 double_6|POINT(1 1.2)
 double_7|POINT(1 1.23)
 double_8|POINT(1 1)
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
 double_15|POINT(1 100)
 double_16|POINT(1 100)
 double_17|POINT(1 0.01)
@@ -422,13 +422,13 @@ double_18|POINT(1 0.01)
 double_19|POINT(1 123)
 double_20|POINT(1 123)
 double_21|POINT(1 -123)
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
-ERROR:  invalid GML representation
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
+ERROR:  invalid GML representation (lwgeom_pg.c:128)
