diff --git a/regress/regress_proj_expected b/regress/regress_proj_expected
index 26412c093..2a201d8b8 100644
--- a/regress/regress_proj_expected
+++ b/regress/regress_proj_expected
@@ -5,7 +5,7 @@
 4|SRID=100001;LINESTRING(574600 5316780,573140 5427940)
 5|SRID=100001;LINESTRING(574600 5316780 0 0,573140 5427940 0 0)
 6|16.00000000|48.00000000
-ERROR:  Input geometry has unknown (0) SRID
+ERROR:  Input geometry has unknown (0) SRID (lwgeom_transform.c:69)
 8|SRID=100002;POINT(0 0)
 9|POINT(574600 5316780)
 10|POINT(574600 5316780)
