diff --git a/regress/setpoint_expected b/regress/setpoint_expected
index 210195c40..3d165608c 100644
--- a/regress/setpoint_expected
+++ b/regress/setpoint_expected
@@ -1,8 +1,8 @@
-ERROR:  abs(Point index) out of range (-)(0..2)
+ERROR:  abs(Point index) out of range (-)(0..2) (lwgeom_functions_basic.c:2314)
 LINESTRING(0 0,1 1,9 9)
-ERROR:  abs(Point index) out of range (-)(0..2)
-ERROR:  First argument must be a LINESTRING
-ERROR:  Third argument must be a POINT
+ERROR:  abs(Point index) out of range (-)(0..2) (lwgeom_functions_basic.c:2314)
+ERROR:  First argument must be a LINESTRING (lwgeom_functions_basic.c:2305)
+ERROR:  Third argument must be a POINT (lwgeom_functions_basic.c:2294)
 LINESTRING(90 91 92,1 1 1,2 2 2)
 LINESTRINGM(0 0 0,90 91 92,2 2 2)
 LINESTRINGM(0 0 0,1 1 1,90 91 93)
