diff --git a/liblwgeom/effectivearea.h b/liblwgeom/effectivearea.h
index 01d023f9a..82f3aed66 100644
--- a/liblwgeom/effectivearea.h
+++ b/liblwgeom/effectivearea.h
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2014 Nicklas Avén
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
