diff --git a/liblwgeom/liblwgeom_topo_internal.h b/liblwgeom/liblwgeom_topo_internal.h
index bd48be8da..8b39030cb 100644
--- a/liblwgeom/liblwgeom_topo_internal.h
+++ b/liblwgeom/liblwgeom_topo_internal.h
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright (C) 2015 Sandro Santilli <strk@kbt.io>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
