diff --git a/liblwgeom/lwchaikins.c b/liblwgeom/lwchaikins.c
index 4312a8631..306a4fe1d 100644
--- a/liblwgeom/lwchaikins.c
+++ b/liblwgeom/lwchaikins.c
@@ -20,6 +20,7 @@
  **********************************************************************
  *
  * Copyright 2018 Nicklas Avén
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
