diff --git a/liblwgeom/lwgeom_log.h b/liblwgeom/lwgeom_log.h
index feb2ac5d7..40203bf20 100644
--- a/liblwgeom/lwgeom_log.h
+++ b/liblwgeom/lwgeom_log.h
@@ -22,6 +22,7 @@
  * Copyright 2008 Paul Ramsey <pramsey@cleverelephant.ca>
  * Copyright 2007-2008 Mark Cave-Ayland
  * Copyright 2001-2006 Refractions Research Inc.
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
