diff --git a/liblwgeom/lwgeom_sfcgal.h b/liblwgeom/lwgeom_sfcgal.h
index 30d359a1b..1bd1e80b9 100644
--- a/liblwgeom/lwgeom_sfcgal.h
+++ b/liblwgeom/lwgeom_sfcgal.h
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2012-2013 Oslandia <infos@oslandia.com>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
