diff --git a/liblwgeom/lwgeodetic.c b/liblwgeom/lwgeodetic.c
index c5dfb8b42..82b785eae 100644
--- a/liblwgeom/lwgeodetic.c
+++ b/liblwgeom/lwgeodetic.c
@@ -20,6 +20,7 @@
  *
  * Copyright 2009 Paul Ramsey <pramsey@cleverelephant.ca>
  * Copyright 2009 David Skea <David.Skea@gov.bc.ca>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
