diff --git a/liblwgeom/lwin_wkt.h b/liblwgeom/lwin_wkt.h
index 14f54db26..bf524aed5 100644
--- a/liblwgeom/lwin_wkt.h
+++ b/liblwgeom/lwin_wkt.h
@@ -20,6 +20,7 @@
  *
  * Copyright (C) 2010-2015 Paul Ramsey <pramsey@cleverelephant.ca>
  * Copyright (C) 2011      Sandro Santilli <strk@kbt.io>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
