diff --git a/liblwgeom/lwin_wkt.c b/liblwgeom/lwin_wkt.c
index 704475d52..5056c87dc 100644
--- a/liblwgeom/lwin_wkt.c
+++ b/liblwgeom/lwin_wkt.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright (C) 2010 Paul Ramsey <pramsey@cleverelephant.ca>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
