diff --git a/liblwgeom/lwiterator.c b/liblwgeom/lwiterator.c
index 55cb0d1a1..4c5cbf1eb 100644
--- a/liblwgeom/lwiterator.c
+++ b/liblwgeom/lwiterator.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2015 Daniel Baston <dbaston@gmail.com>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
