diff --git a/liblwgeom/lwkmeans.c b/liblwgeom/lwkmeans.c
index cbcb28f94..52aa588ea 100644
--- a/liblwgeom/lwkmeans.c
+++ b/liblwgeom/lwkmeans.c
@@ -2,6 +2,7 @@
  *
  * Copyright (c) 2018, Darafei Praliaskouski <me@komzpa.net>
  * Copyright (c) 2016, Paul Ramsey <pramsey@cleverelephant.ca>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  *------------------------------------------------------------------------*/
 
