diff --git a/liblwgeom/lwmcurve.c b/liblwgeom/lwmcurve.c
index e797992ba..003346dc8 100644
--- a/liblwgeom/lwmcurve.c
+++ b/liblwgeom/lwmcurve.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2001-2006 Refractions Research Inc.
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
