diff --git a/liblwgeom/lwout_wkb.c b/liblwgeom/lwout_wkb.c
index 4cf4cd319..6ae2c62f0 100644
--- a/liblwgeom/lwout_wkb.c
+++ b/liblwgeom/lwout_wkb.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright (C) 2009 Paul Ramsey <pramsey@cleverelephant.ca>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
