diff --git a/liblwgeom/lwout_x3d.h b/liblwgeom/lwout_x3d.h
index 31a35151c..6b0a45307 100644
--- a/liblwgeom/lwout_x3d.h
+++ b/liblwgeom/lwout_x3d.h
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2011-2017 Arrival 3D, Regina Obe
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
