diff --git a/liblwgeom/lwtriangle.c b/liblwgeom/lwtriangle.c
index dfe490a11..9b6ac7694 100644
--- a/liblwgeom/lwtriangle.c
+++ b/liblwgeom/lwtriangle.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright (C) 2010 - Oslandia
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
