diff --git a/liblwgeom/Makefile.in b/liblwgeom/Makefile.in
index 265302c2c..eac8fa841 100644
--- a/liblwgeom/Makefile.in
+++ b/liblwgeom/Makefile.in
@@ -19,12 +19,13 @@
 # **********************************************************************
 # *
 # * Copyright 2008 Mark Cave-Ayland
+# * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 # *
 # **********************************************************************/
 
 CC = @CC@
 CPPFLAGS = @CPPFLAGS@
-CFLAGS = @CFLAGS@ @PICFLAGS@ @WARNFLAGS@ @GEOS_CPPFLAGS@ @PROJ_CPPFLAGS@ @JSON_CPPFLAGS@
+CFLAGS = @CFLAGS@ @PICFLAGS@ @PGSQL_BE_CPPFLAGS@ @WARNFLAGS@ @GEOS_CPPFLAGS@ @PROJ_CPPFLAGS@ @JSON_CPPFLAGS@
 LDFLAGS = @LDFLAGS@ @GEOS_LDFLAGS@ -lgeos_c @PROJ_LDFLAGS@ -lproj @JSON_LDFLAGS@ -lm
 NUMERICFLAGS = @NUMERICFLAGS@
 top_builddir = @top_builddir@
