diff --git a/liblwgeom/measures3d.h b/liblwgeom/measures3d.h
index 84d1c2f74..d316da42f 100644
--- a/liblwgeom/measures3d.h
+++ b/liblwgeom/measures3d.h
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2011 Nicklas Avén
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
