diff --git a/liblwgeom/varint.c b/liblwgeom/varint.c
index 22b8e7ef6..6b45cd2dd 100644
--- a/liblwgeom/varint.c
+++ b/liblwgeom/varint.c
@@ -20,6 +20,7 @@
  *
  * Copyright (C) 2014 Sandro Santilli <strk@kbt.io>
  * Copyright (C) 2013 Nicklas Avén
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
