diff --git a/topology/sql/polygonize.sql.in b/topology/sql/polygonize.sql.in
index ad2db14bf..9abbb4ebf 100644
--- a/topology/sql/polygonize.sql.in
+++ b/topology/sql/polygonize.sql.in
@@ -4,6 +4,7 @@
 -- http://postgis.net
 --
 -- Copyright (C) 2011 Sandro Santilli <strk@kbt.io>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 -- This is free software; you can redistribute and/or modify it under
 -- the terms of the GNU General Public Licence. See the COPYING file.
