diff --git a/extras/tiger_geocoder/regress/normalize_address_regress b/extras/tiger_geocoder/regress/normalize_address_regress
index 6a417acaa..c7f7766a7 100644
--- a/extras/tiger_geocoder/regress/normalize_address_regress
+++ b/extras/tiger_geocoder/regress/normalize_address_regress
@@ -43,3 +43,5 @@
 #1614a|8435 Co Rd 20 SE, ROCHESTER, MN 55904|8435||20|Co Rd|SE||ROCHESTER|MN|55904|t||8435
 #1614b|3208 US Hwy 52, Rochester, MN 55901|3208||52|US Hwy|||Rochester|MN|55901|t||3208
 #1108a|529 Main St, Suite 201, Boston, MA 02129|529||Main|St||Suite 201|Boston|MA|02129|t||529
+#GPN001a|1017 , APT 12, ST PAUL, MN 55105|1017|||ST PAUL|MN
+#GPN001a|1029 ATLANTIC ST APT 302 St, APT 302, ST PAUL, MN 55106|1029||ATLANTIC ST APT 302 St|ST PAUL|MN
