diff --git a/extras/tiger_geocoder/regress/pagc_normalize_address_regress b/extras/tiger_geocoder/regress/pagc_normalize_address_regress
index 8514d9e07..f8b8477c5 100644
--- a/extras/tiger_geocoder/regress/pagc_normalize_address_regress
+++ b/extras/tiger_geocoder/regress/pagc_normalize_address_regress
@@ -50,4 +50,6 @@ true
 #2978a|10||DORRANCE|ST|||PROVIDENCE|RI||t||10 20
 #2978b|10||DORRANCE|ST|||PROVIDENCE|RI||t||10 20
 #2978c|10||DORRANCE|ST|||PROVIDENCE|RI||t||10 20
+#GPP001a|1017 LINWOOD AVE, APT PAUL, MN 12 ST 55105|1017||LINWOOD AVE|APT PAUL|MN
+#GPP001a|1029 ATLANTIC ST, APT PAUL, MN 302 ST 55106|1029||ATLANTIC ST|APT PAUL|MN
 false
