diff --git a/loader/cunit/cu_tester.h b/loader/cunit/cu_tester.h
index 18707b6d5..ac03a1dc2 100644
--- a/loader/cunit/cu_tester.h
+++ b/loader/cunit/cu_tester.h
@@ -3,6 +3,7 @@
  * PostGIS - Spatial Types for PostgreSQL
  * http://postgis.net
  * Copyright 2010 Mark Leslie
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  * This is free software; you can redistribute and/or modify it under
  * the terms of the GNU General Public Licence. See the COPYING file.
