diff --git a/configure.ac b/configure.ac
index 915030af4..e6798add2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -3,6 +3,7 @@ dnl *
 dnl * PostGIS - Spatial Types for PostgreSQL
 dnl * http://postgis.net
 dnl * Copyright 2008 Mark Cave-Ayland
+dnl * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 dnl *
 dnl * This is free software; you can redistribute and/or modify it under
 dnl * the terms of the GNU General Public Licence. See the COPYING file.
@@ -714,7 +715,7 @@ GEOS_NUMERIC_VERSION="$GEOS_MAJOR_VERSION$GEOS_NUMERIC_MINOR_VERSION$GEOS_NUMERI
 
 dnl Ensure that we are using GEOS >= 3.5.0
 AC_MSG_RESULT([checking GEOS version... $GEOS_FULL_VERSION])
-if test ! "$GEOS_NUMERIC_VERSION" -ge 30500; then
+if test ! "$GEOS_NUMERIC_VERSION" -ge 30300; then
 	AC_MSG_ERROR([PostGIS requires GEOS >= 3.5.0])
 fi
 
