diff --git a/doc/xsl/post_gis_day_cards.html.xsl b/doc/xsl/post_gis_day_cards.html.xsl
index 88efd8258..5eceb6ca1 100644
--- a/doc/xsl/post_gis_day_cards.html.xsl
+++ b/doc/xsl/post_gis_day_cards.html.xsl
@@ -3,6 +3,7 @@
 <!-- ********************************************************************
      ********************************************************************
 	 Copyright 2011, Regina Obe
+	 Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
      License: BSD
 	 Purpose: This is an xsl transform that generates PostGIS flash cards
      ******************************************************************** -->
