diff --git a/doc/xsl/topology_gardentest.sql.xsl b/doc/xsl/topology_gardentest.sql.xsl
index f623225a4..7abe58691 100644
--- a/doc/xsl/topology_gardentest.sql.xsl
+++ b/doc/xsl/topology_gardentest.sql.xsl
@@ -3,6 +3,7 @@
 <!-- ********************************************************************
  ********************************************************************
 	 Copyright 2008-2010, Regina Obe
+	 Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 	 License: BSD
 	 Purpose: This is an xsl transform that generates an sql test script from xml docs to test all the functions we have documented
 			using a garden variety of geometries.  Its intent is to flag major crashes.
