diff --git a/topology/sql/query/getfacebypoint.sql.in b/topology/sql/query/getfacebypoint.sql.in
index 5b53cd29c..839926534 100644
--- a/topology/sql/query/getfacebypoint.sql.in
+++ b/topology/sql/query/getfacebypoint.sql.in
@@ -4,6 +4,7 @@
 -- http://postgis.net
 --
 -- Copyright (C) 2011 Andrea Peri <aperi2007@gmail.com>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 -- This is free software; you can redistribute and/or modify it under
 -- the terms of the GNU General Public Licence. See the COPYING file.
