diff --git a/raster/test/cunit/cu_band_basics.c b/raster/test/cunit/cu_band_basics.c
index 836e66145..d627ad075 100644
--- a/raster/test/cunit/cu_band_basics.c
+++ b/raster/test/cunit/cu_band_basics.c
@@ -5,6 +5,7 @@
  * Copyright (C) 2018 Bborie Park <dustymugs@gmail.com>
  * Copyright (C) 2012 Regents of the University of California
  *   <bkpark@ucdavis.edu>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License
