diff --git a/raster/test/cunit/cu_raster_geometry.c b/raster/test/cunit/cu_raster_geometry.c
index 9253a3d53..fc82f88f5 100644
--- a/raster/test/cunit/cu_raster_geometry.c
+++ b/raster/test/cunit/cu_raster_geometry.c
@@ -4,6 +4,7 @@
  *
  * Copyright (C) 2012-2013 Regents of the University of California
  *   <bkpark@ucdavis.edu>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License
@@ -258,9 +259,9 @@ static void test_raster_surface() {
 	const int maxX = 5;
 	const int maxY = 5;
 	int x, y;
-	char *wkt = NULL;
 	LWMPOLY *mpoly = NULL;
 	int err;
+	LWGEOM *gexpected, *gobserved;
 
 	rast = rt_raster_new(maxX, maxY);
 	CU_ASSERT(rast != NULL);
@@ -280,9 +281,10 @@ static void test_raster_surface() {
 	err = rt_raster_surface(rast, 0, &mpoly);
 	CU_ASSERT_EQUAL(err, ES_NONE);
 	CU_ASSERT(mpoly != NULL);
-	wkt = lwgeom_to_text(lwmpoly_as_lwgeom(mpoly));
-	CU_ASSERT_STRING_EQUAL(wkt, "MULTIPOLYGON(((0 0,0 -5,5 -5,5 0,0 0)))");
-	rtdealloc(wkt);
+	//wkt = lwgeom_to_text(lwmpoly_as_lwgeom(mpoly));
+	gobserved = (LWGEOM *)lwmpoly_as_lwgeom(mpoly);
+	gexpected = lwgeom_from_wkt("MULTIPOLYGON(((0 0,0 -5,5 -5,5 0,0 0)))", LW_PARSER_CHECK_NONE);
+	CU_ASSERT_DOUBLE_EQUAL(lwgeom_area(gobserved), lwgeom_area(gexpected), FLT_EPSILON);
 	lwmpoly_free(mpoly);
 	mpoly = NULL;
 
@@ -292,9 +294,9 @@ static void test_raster_surface() {
 	err = rt_raster_surface(rast, 0, &mpoly);
 	CU_ASSERT_EQUAL(err, ES_NONE);
 	CU_ASSERT(mpoly != NULL);
-	wkt = lwgeom_to_text(lwmpoly_as_lwgeom(mpoly));
-	CU_ASSERT_STRING_EQUAL(wkt, "MULTIPOLYGON(((1 0,1 -1,0 -1,0 -5,4 -5,5 -5,5 0,1 0)))");
-	rtdealloc(wkt);
+	gobserved = (LWGEOM *)lwmpoly_as_lwgeom(mpoly);
+	gexpected = lwgeom_from_wkt("MULTIPOLYGON(((1 0,1 -1,0 -1,0 -5,4 -5,5 -5,5 0,1 0)))", LW_PARSER_CHECK_NONE);
+	CU_ASSERT_DOUBLE_EQUAL(lwgeom_area(gobserved), lwgeom_area(gexpected), FLT_EPSILON);
 	lwmpoly_free(mpoly);
 	mpoly = NULL;
 
@@ -304,9 +306,10 @@ static void test_raster_surface() {
 	err = rt_raster_surface(rast, 0, &mpoly);
 	CU_ASSERT_EQUAL(err, ES_NONE);
 	CU_ASSERT(mpoly != NULL);
-	wkt = lwgeom_to_text(lwmpoly_as_lwgeom(mpoly));
-	CU_ASSERT_STRING_EQUAL(wkt, "MULTIPOLYGON(((1 0,1 -1,0 -1,0 -5,4 -5,5 -5,5 0,1 0),(1 -1,1 -2,2 -2,2 -1,1 -1)))");
-	rtdealloc(wkt);
+	gobserved = (LWGEOM *)lwmpoly_as_lwgeom(mpoly);
+	gexpected = lwgeom_from_wkt("MULTIPOLYGON(((1 0,1 -1,0 -1,0 -5,4 -5,5 -5,5 0,1 0),(1 -1,1 -2,2 -2,2 -1,1 -1)))",
+                                LW_PARSER_CHECK_NONE);
+	CU_ASSERT_DOUBLE_EQUAL(lwgeom_area(gobserved), lwgeom_area(gexpected), FLT_EPSILON);
 	lwmpoly_free(mpoly);
 	mpoly = NULL;
 
@@ -316,11 +319,11 @@ static void test_raster_surface() {
 	err = rt_raster_surface(rast, 0, &mpoly);
 	CU_ASSERT_EQUAL(err, ES_NONE);
 	CU_ASSERT(mpoly != NULL);
-	wkt = lwgeom_to_text(lwmpoly_as_lwgeom(mpoly));
-
-	CU_ASSERT_STRING_EQUAL(wkt, "MULTIPOLYGON(((1 -1,1 0,5 0,5 -5,4 -5,0 -5,0 -1,1 -1),(1 -1,1 -2,2 -2,2 -1,1 -1),(2 -2,2 -3,3 -3,3 -2,2 -2)))");
-
-	rtdealloc(wkt);
+	gobserved = (LWGEOM *)lwmpoly_as_lwgeom(mpoly);
+	gexpected = lwgeom_from_wkt(
+	        "MULTIPOLYGON(((1 -1,1 0,5 0,5 -5,4 -5,0 -5,0 -1,1 -1),(1 -1,1 -2,2 -2,2 -1,1 -1),(2 -2,2 -3,3 -3,3 -2,2 -2)))",
+	        LW_PARSER_CHECK_NONE);
+	CU_ASSERT_DOUBLE_EQUAL(lwgeom_area(gobserved), lwgeom_area(gexpected), FLT_EPSILON);
 	lwmpoly_free(mpoly);
 	mpoly = NULL;
 
@@ -330,11 +333,11 @@ static void test_raster_surface() {
 	err = rt_raster_surface(rast, 0, &mpoly);
 	CU_ASSERT_EQUAL(err, ES_NONE);
 	CU_ASSERT(mpoly != NULL);
-	wkt = lwgeom_to_text(lwmpoly_as_lwgeom(mpoly));
-
-	CU_ASSERT_STRING_EQUAL(wkt, "MULTIPOLYGON(((1 -1,1 0,5 0,5 -5,4 -5,0 -5,0 -1,1 -1),(1 -1,1 -2,2 -2,2 -1,1 -1),(2 -2,2 -3,3 -3,3 -2,2 -2),(3 -3,3 -4,4 -4,4 -3,3 -3)))");
-
-	rtdealloc(wkt);
+	gobserved = (LWGEOM *)lwmpoly_as_lwgeom(mpoly);
+	gexpected = lwgeom_from_wkt(
+	        "MULTIPOLYGON(((1 -1,1 0,5 0,5 -5,4 -5,0 -5,0 -1,1 -1),(1 -1,1 -2,2 -2,2 -1,1 -1),(2 -2,2 -3,3 -3,3 -2,2 -2),(3 -3,3 -4,4 -4,4 -3,3 -3)))",
+	        LW_PARSER_CHECK_NONE);
+	CU_ASSERT_DOUBLE_EQUAL(lwgeom_area(gobserved), lwgeom_area(gexpected), FLT_EPSILON);
 	lwmpoly_free(mpoly);
 	mpoly = NULL;
 
@@ -344,9 +347,11 @@ static void test_raster_surface() {
 	err = rt_raster_surface(rast, 0, &mpoly);
 	CU_ASSERT_EQUAL(err, ES_NONE);
 	CU_ASSERT(mpoly != NULL);
-	wkt = lwgeom_to_text(lwmpoly_as_lwgeom(mpoly));
-	CU_ASSERT_STRING_EQUAL(wkt, "MULTIPOLYGON(((4 -4,4 -5,0 -5,0 -1,1 -1,1 -2,2 -2,2 -3,3 -3,3 -4,4 -4)),((1 -1,1 0,5 0,5 -4,4 -4,4 -3,3 -3,3 -2,2 -2,2 -1,1 -1)))");
-	rtdealloc(wkt);
+	gobserved = (LWGEOM *)lwmpoly_as_lwgeom(mpoly);
+	gexpected = lwgeom_from_wkt(
+	        "MULTIPOLYGON(((4 -4,4 -5,0 -5,0 -1,1 -1,1 -2,2 -2,2 -3,3 -3,3 -4,4 -4)),((1 -1,1 0,5 0,5 -4,4 -4,4 -3,3 -3,3 -2,2 -2,2 -1,1 -1)))",
+	        LW_PARSER_CHECK_NONE);
+	CU_ASSERT_DOUBLE_EQUAL(lwgeom_area(gobserved), lwgeom_area(gexpected), FLT_EPSILON);
 	lwmpoly_free(mpoly);
 	mpoly = NULL;
 
@@ -359,9 +364,11 @@ static void test_raster_surface() {
 	err = rt_raster_surface(rast, 0, &mpoly);
 	CU_ASSERT_EQUAL(err, ES_NONE);
 	CU_ASSERT(mpoly != NULL);
-	wkt = lwgeom_to_text(lwmpoly_as_lwgeom(mpoly));
-	CU_ASSERT_STRING_EQUAL(wkt, "MULTIPOLYGON(((1 -4,2 -4,2 -3,3 -3,3 -4,4 -4,4 -5,3 -5,1 -5,1 -4)),((1 -4,0 -4,0 -1,1 -1,1 -2,2 -2,2 -3,1 -3,1 -4)),((3 -2,4 -2,4 -1,5 -1,5 -4,4 -4,4 -3,3 -3,3 -2)),((3 -2,2 -2,2 -1,1 -1,1 0,4 0,4 -1,3 -1,3 -2)))");
-	rtdealloc(wkt);
+	gobserved = (LWGEOM *)lwmpoly_as_lwgeom(mpoly);
+	gexpected = lwgeom_from_wkt(
+	        "MULTIPOLYGON(((1 -4,2 -4,2 -3,3 -3,3 -4,4 -4,4 -5,3 -5,1 -5,1 -4)),((1 -4,0 -4,0 -1,1 -1,1 -2,2 -2,2 -3,1 -3,1 -4)),((3 -2,4 -2,4 -1,5 -1,5 -4,4 -4,4 -3,3 -3,3 -2)),((3 -2,2 -2,2 -1,1 -1,1 0,4 0,4 -1,3 -1,3 -2)))",
+	        LW_PARSER_CHECK_NONE);
+	CU_ASSERT_DOUBLE_EQUAL(lwgeom_area(gobserved), lwgeom_area(gexpected), FLT_EPSILON);
 	lwmpoly_free(mpoly);
 	mpoly = NULL;
 
