diff --git a/extras/tiger_geocoder/utility/set_search_path.sql b/extras/tiger_geocoder/utility/set_search_path.sql
index c8e8595bb..4e353c6c7 100644
--- a/extras/tiger_geocoder/utility/set_search_path.sql
+++ b/extras/tiger_geocoder/utility/set_search_path.sql
@@ -1,6 +1,7 @@
  /***
  *
  * Copyright (C) 2012 Regina Obe and Leo Hsu (Paragon Corporation)
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  **/
 -- Adds a schema to  the front of search path so that functions, tables etc get installed by default in set schema
 -- but if people have postgis and other things installed in non-public, it will still keep those in path
