diff --git a/utils/postgis_proc_set_search_path.pl b/utils/postgis_proc_set_search_path.pl
index 8425787b6..3cea71765 100644
--- a/utils/postgis_proc_set_search_path.pl
+++ b/utils/postgis_proc_set_search_path.pl
@@ -5,6 +5,7 @@
 # http://postgis.net
 #
 # Copyright (C) 2016 Regina Obe <lr@pcorp.us>
+# Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 #
 # This is free software; you can redistribute and/or modify it under
 # the terms of the GNU General Public Licence. See the COPYING file.
