diff --git a/topology/sql/topogeometry/topogeom_edit.sql.in b/topology/sql/topogeometry/topogeom_edit.sql.in
index 0889bb762..4bd9912ca 100644
--- a/topology/sql/topogeometry/topogeom_edit.sql.in
+++ b/topology/sql/topogeometry/topogeom_edit.sql.in
@@ -4,6 +4,7 @@
 -- http://postgis.net
 --
 -- Copyright (C) 2015 Sandro Santilli <strk@kbt.io>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 -- This is free software; you can redistribute and/or modify it under
 -- the terms of the GNU General Public Licence. See the COPYING file.
