diff --git a/extensions/address_standardizer/COPYING b/extensions/address_standardizer/COPYING
index 64129a105..79fa3f55c 100644
--- a/extensions/address_standardizer/COPYING
+++ b/extensions/address_standardizer/COPYING
@@ -3,6 +3,7 @@ Copyright (c) 2008 Walter Bruce Sinclair
 
 woodbri@swoodbridge.com
 woodbr@imaptools.com
+Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 
 Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
 
