diff --git a/extensions/address_standardizer/std_pg_hash.c b/extensions/address_standardizer/std_pg_hash.c
index f12786051..bb505346f 100644
--- a/extensions/address_standardizer/std_pg_hash.c
+++ b/extensions/address_standardizer/std_pg_hash.c
@@ -638,7 +638,7 @@ static int parse_rule(char *buf, int *rule)
     TRGT = DatumGetInt32(binval);
 
 #define GET_TEXT_FROM_TUPLE(TRGT,WHICH) \
-    TRGT = DatumGetCString(SPI_getvalue(tuple, tupdesc, WHICH));
+    TRGT = SPI_getvalue(tuple, tupdesc, WHICH);
 
 
 static int fetch_lex_columns(SPITupleTable *tuptable, lex_columns_t *lex_cols)
