diff --git a/postgis/geography_centroid.c b/postgis/geography_centroid.c
index 411bd914d..55fb82868 100644
--- a/postgis/geography_centroid.c
+++ b/postgis/geography_centroid.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright (C) 2017 Danny Götte <danny.goette@fem.tu-ilmenau.de>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
