diff --git a/postgis/gserialized_gist_nd.c b/postgis/gserialized_gist_nd.c
index 44540a685..e07d238bc 100644
--- a/postgis/gserialized_gist_nd.c
+++ b/postgis/gserialized_gist_nd.c
@@ -20,6 +20,7 @@
  *
  * Copyright 2009 Paul Ramsey <pramsey@cleverelephant.ca>
  * Copyright 2017 Darafei Praliaskouski <me@komzpa.net>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
@@ -184,7 +185,8 @@ void gidx_merge(GIDX **b_union, GIDX *b_new)
 	/* Merge of unknown and known is known */
 	if( gidx_is_unknown(*b_union) )
 	{
-		*b_union = b_new;
+		pfree(*b_union);
+		*b_union = gidx_copy(b_new);
 		return;
 	}
 
