diff --git a/postgis/gserialized_gist_2d.c b/postgis/gserialized_gist_2d.c
index a44680aba..0f7f9f435 100644
--- a/postgis/gserialized_gist_2d.c
+++ b/postgis/gserialized_gist_2d.c
@@ -20,6 +20,7 @@
  *
  * Copyright 2009 Paul Ramsey <pramsey@cleverelephant.ca>
  * Copyright 2017 Darafei Praliaskouski <me@komzpa.net>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
