diff --git a/postgis/lwgeom_rtree.c b/postgis/lwgeom_rtree.c
index e30b23761..3a3a129c5 100644
--- a/postgis/lwgeom_rtree.c
+++ b/postgis/lwgeom_rtree.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright (C) 2001-2005 Refractions Research Inc.
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
