diff --git a/postgis/lwgeom_in_kml.c b/postgis/lwgeom_in_kml.c
index 8daa58a51..f602e366f 100644
--- a/postgis/lwgeom_in_kml.c
+++ b/postgis/lwgeom_in_kml.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2009 Oslandia
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
