diff --git a/postgis/lwgeom_sfcgal.c b/postgis/lwgeom_sfcgal.c
index 5660486d7..913315c1c 100644
--- a/postgis/lwgeom_sfcgal.c
+++ b/postgis/lwgeom_sfcgal.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2012-2013 Oslandia <infos@oslandia.com>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
