diff --git a/postgis/lwgeom_sfcgal.h b/postgis/lwgeom_sfcgal.h
index c350ba65f..713fc49d2 100644
--- a/postgis/lwgeom_sfcgal.h
+++ b/postgis/lwgeom_sfcgal.h
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2012-2013 Oslandia <infos@oslandia.com>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
