diff --git a/postgis/lwgeom_export.h b/postgis/lwgeom_export.h
index e9652a0aa..1125ad2fd 100644
--- a/postgis/lwgeom_export.h
+++ b/postgis/lwgeom_export.h
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2009 Olivier Courtin <olivier.courtin@oslandia.com>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
