diff --git a/postgis/lwgeom_export.c b/postgis/lwgeom_export.c
index 4bc16fa00..156faedc8 100644
--- a/postgis/lwgeom_export.c
+++ b/postgis/lwgeom_export.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2009-2011 Olivier Courtin <olivier.courtin@oslandia.com>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
@@ -39,6 +40,7 @@
 #include "lwgeom_pg.h"
 #include "liblwgeom.h"
 #include "lwgeom_export.h"
+#include "libsrid.h"
 
 Datum LWGEOM_asGML(PG_FUNCTION_ARGS);
 Datum LWGEOM_asKML(PG_FUNCTION_ARGS);
@@ -61,6 +63,17 @@ char * getSRSbySRID(int srid, bool short_crs)
 	char *srs, *srscopy;
 	int size, err;
 
+    /* In GPDB, we don't support the SRID retrieving from spatial_ref_sys,
+     *  otherwise we will meet the known issue: cannot access relation from segments.
+     *  so we search static hash table firstly to by-pass this issue issue.
+     */
+    if (getSRSbySRIDbyRule(srid, short_crs, query) != NULL) {
+        size = strlen(query) + 1;
+        srscopy = SPI_palloc(size);
+        memcpy(srscopy, query, size);
+        return srscopy;
+    }
+
 	if (SPI_OK_CONNECT != SPI_connect ())
 	{
 		elog(NOTICE, "getSRSbySRID: could not connect to SPI manager");
@@ -128,6 +141,10 @@ int getSRIDbySRS(const char* srs)
 	Datum values[] = {CStringGetDatum(srs)};
 	int32_t srid, err;
 
+	srid = getSRIDbySRSbyRule(srs);
+    if (srid != 0) {
+        return srid;
+    }
 	if (!srs) return 0;
 
 	if (SPI_OK_CONNECT != SPI_connect())
