diff --git a/postgis/lwgeom_functions_analytic.h b/postgis/lwgeom_functions_analytic.h
index 10cb73b04..f49b796fe 100644
--- a/postgis/lwgeom_functions_analytic.h
+++ b/postgis/lwgeom_functions_analytic.h
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2001-2011 Refractions Research Inc.
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
