diff --git a/postgis/lwgeom_ogc.c b/postgis/lwgeom_ogc.c
index 2660296dc..37e1d24a3 100644
--- a/postgis/lwgeom_ogc.c
+++ b/postgis/lwgeom_ogc.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2001-2005 Refractions Research Inc.
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
