diff --git a/postgis/lwgeom_geos.h b/postgis/lwgeom_geos.h
index f204a801d..55ddb9eba 100644
--- a/postgis/lwgeom_geos.h
+++ b/postgis/lwgeom_geos.h
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2008 Paul Ramsey <pramsey@cleverelephant.ca>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
