diff --git a/postgis/lwgeom_accum.c b/postgis/lwgeom_accum.c
index 0d2d80efc..3f7ce2b4b 100644
--- a/postgis/lwgeom_accum.c
+++ b/postgis/lwgeom_accum.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2009 Paul Ramsey <pramsey@opengeo.org>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
@@ -361,15 +362,12 @@ pgis_geometry_clusterwithin_finalfn(PG_FUNCTION_ARGS)
 
 	if (PG_ARGISNULL(0))
 		PG_RETURN_NULL();
-
 	p = (pgis_abs*) PG_GETARG_POINTER(0);
 
 	if (!p->data)
 	{
-		elog(ERROR, "Tolerance not defined");
 		PG_RETURN_NULL();
 	}
-
 	geometry_array = pgis_accum_finalfn(p, CurrentMemoryContext, fcinfo);
 	result = PGISDirectFunctionCall2( cluster_within_distance_garray, geometry_array, p->data);
 	if (!result)
@@ -439,12 +437,12 @@ PGISDirectFunctionCall2(PGFunction func, Datum arg1, Datum arg2)
 	fcinfo.arg[1] = arg2;
 	fcinfo.argnull[0] = false;
 	fcinfo.argnull[1] = false;
-
 	result = (*func) (&fcinfo);
 
 	/* Check for null result, returning a "NULL" Datum if indicated */
-	if (fcinfo.isnull)
+	if (fcinfo.isnull){
 		return (Datum) 0;
+	}
 
 	return result;
 #else
