diff --git a/postgis/lwgeom_in_encoded_polyline.c b/postgis/lwgeom_in_encoded_polyline.c
index 6ebd9da18..26c3b6482 100644
--- a/postgis/lwgeom_in_encoded_polyline.c
+++ b/postgis/lwgeom_in_encoded_polyline.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
 * Copyright 2014 Kashif Rasul <kashif.rasul@gmail.com> and
+* Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
