diff --git a/postgis/lwgeom_triggers.c b/postgis/lwgeom_triggers.c
index 9df48c9af..e5a38e88a 100644
--- a/postgis/lwgeom_triggers.c
+++ b/postgis/lwgeom_triggers.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright (C) 2004 Refractions Research Inc.
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
