diff --git a/postgis/mvt.h b/postgis/mvt.h
index 261d81117..a6c644b33 100644
--- a/postgis/mvt.h
+++ b/postgis/mvt.h
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright (C) 2016-2017 Björn Harrtell <bjorn@wololo.org>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
