diff --git a/postgis/uninstall_sfcgal.sql.in b/postgis/uninstall_sfcgal.sql.in
index 095131f15..a5cc00609 100644
--- a/postgis/uninstall_sfcgal.sql.in
+++ b/postgis/uninstall_sfcgal.sql.in
@@ -2,6 +2,7 @@
 --
 -- PostGIS - SFCGAL functions
 -- Copyright 2012-2013 Oslandia <infos@oslandia.com>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 -- This is free software; you can redistribute and/or modify it under
 -- the terms of the GNU General Public Licence. See the COPYING file.
