diff --git a/raster/test/cunit/Makefile.in b/raster/test/cunit/Makefile.in
index 4124c630c..de1407138 100644
--- a/raster/test/cunit/Makefile.in
+++ b/raster/test/cunit/Makefile.in
@@ -3,6 +3,7 @@
 # * PostGIS - Spatial Types for PostgreSQL
 # * http://postgis.net
 # * Copyright 2008 Paul Ramsey, Mark Cave-Ayland
+# * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 # *
 # * This is free software; you can redistribute and/or modify it under
 # * the terms of the GNU General Public Licence. See the COPYING file.
