diff --git a/raster/rt_core/Makefile.in b/raster/rt_core/Makefile.in
index d5a8f5d58..44d0c18be 100644
--- a/raster/rt_core/Makefile.in
+++ b/raster/rt_core/Makefile.in
@@ -1,6 +1,7 @@
 #############################################################################
 #
 # Copyright (c) 2009 Sandro Santilli <strk@kbt.io>
+# Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 #
 # This program is free software; you can redistribute it and/or
 # modify it under the terms of the GNU General Public License
