diff --git a/raster/test/Makefile.in b/raster/test/Makefile.in
index a4196f54f..1e81758c5 100644
--- a/raster/test/Makefile.in
+++ b/raster/test/Makefile.in
@@ -1,6 +1,7 @@
 #############################################################################
 #
 # Copyright (c) 2009 Mateusz Loskot <mateusz@loskot.net>
+# Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 #
 # This is free software; you can redistribute and/or modify it under
 # the terms of the GNU General Public Licence. See the COPYING file.
