diff --git a/regress/clean_expected b/regress/clean_expected
index d3159135f..307d75b61 100644
--- a/regress/clean_expected
+++ b/regress/clean_expected
@@ -1,3 +1,8 @@
+PG|1|t|t|f
+PG|2|t|t|f
+PG|3|t|t|f
+PG|4|t|t|f
+PG|5|t|t|f
 RT|1|t|t|f
 RT|2|t|t|f
 RT|3|t|t|f
@@ -21,11 +26,6 @@ RT|16.2|t|t|f
 RT|16.3|t|t|f
 RT|16.4|t|t|f
 RT|17.1|t|t|f
-PG|1|t|t|f
-PG|2|t|t|f
-PG|3|t|t|f
-PG|4|t|t|f
-PG|5|t|t|f
 #1719.1|POINT(0 0)
-#1719.2|GEOMETRYCOLLECTION(POINT(0 0),MULTIPOINT(3 4,5 2),POINT(4 4),MULTIPOLYGON(((0 0,5 5,10 0,0 0)),((5 5,0 10,10 10,5 5))))
+#1719.2|GEOMETRYCOLLECTION(MULTIPOLYGON(((0 10,10 10,5 5,0 10)),((0 0,5 5,10 0,0 0))),MULTIPOINT(5 2,3 4),POINT(4 4),POINT(0 0))
 #1719.3|MULTIPOINT(3 4,5 2)
