diff --git a/regress/in_geojson_expected b/regress/in_geojson_expected
index 9d48f40a2..5f28d2b0d 100644
--- a/regress/in_geojson_expected
+++ b/regress/in_geojson_expected
@@ -25,8 +25,8 @@ geomfromgeojson_z_01|POINT(1 2 3)
 geomfromgeojson_z_02|LINESTRING(1 2 3,2 3 4)
 geomfromgeojson_zm_01|POINT(1 2 3)
 geomfromgeojson_zm_02|LINESTRING(1 2 3,2 3 4)
-geomfromgeojson_srs_1|GEOMETRYCOLLECTION(POINT(100 0),LINESTRING(101 0,102 1))
-geomfromgeojson_srs_2|GEOMETRYCOLLECTION(POINT(100 0),LINESTRING(101 0,102 1))
+geomfromgeojson_srs_1|SRID=4326;GEOMETRYCOLLECTION(POINT(100 0),LINESTRING(101 0,102 1))
+geomfromgeojson_srs_2|SRID=4326;GEOMETRYCOLLECTION(POINT(100 0),LINESTRING(101 0,102 1))
 geomfromgeojson_srs_3|GEOMETRYCOLLECTION(POINT(100 0),LINESTRING(101 0,102 1))
 geomfromgeojson_srs_4|GEOMETRYCOLLECTION(POINT(100 0),LINESTRING(101 0,102 1))
 #3583|MULTIPOLYGON Z (EMPTY)
