diff --git a/regress/lwgeom_regress_expected b/regress/lwgeom_regress_expected
index 4a22c59a0..0a6169bb1 100644
--- a/regress/lwgeom_regress_expected
+++ b/regress/lwgeom_regress_expected
@@ -23,21 +23,20 @@ BoundingDiagonal5|SRID=3857;LINESTRINGM(4 4 0,5 4 1)
 BoundingDiagonal6|SRID=3857;LINESTRINGM EMPTY
 ST_Azimuth_regular|2.3562
 ST_Azimuth_same_point|
-ERROR:  Operation on mixed SRID geometries
-ERROR:  Argument must be POINT geometries
-ERROR:  parse error - invalid geometry
+ERROR:  Operation on mixed SRID geometries (lwgeom_pg.c:128)
+ERROR:  Argument must be POINT geometries (lwgeom_pg.c:128)
+ERROR:  parse error - invalid geometry (lwgeom_pg.c:58)
 ST_Angle_4_pts|4.71238898038469
 ST_Angle_4_pts|0.785398163397448
 ST_Angle_3_pts|1.5707963267949
-ERROR:  Operation on mixed SRID geometries
-ERROR:  Empty geometry
+ERROR:  Operation on mixed SRID geometries (lwgeom_pg.c:128)
+ERROR:  Empty geometry (lwgeom_pg.c:128)
 ST_Angle_2_lines|4.71238898038469
 #3965|25|25
 #3971|t
 #4071|2|3|4
 typmod_point_4326|(Point,4326)
 typmod_point_0|(Point)
-NOTICE:  SRID value -1 converted to the officially unknown SRID value 0
 typmod_point_-1|(Point)
 typmod_pointzm_0|(PointZM)
 typmod_geometry_0|
