diff --git a/regress/mvt.sql b/regress/mvt.sql
index c09fab9b8..6d09902b1 100644
--- a/regress/mvt.sql
+++ b/regress/mvt.sql
@@ -11,10 +11,10 @@ select 'PG3', ST_AsText(ST_AsMVTGeom(
 	ST_Point(1, 2),
 	ST_MakeBox2D(ST_Point(0, 0), ST_Point(4096/2, 4096/2)),
 	4096, 0, false));
-select 'PG4', ST_AsText(ST_AsMVTGeom(
-	ST_GeomFromText('POLYGON ((0 0, 10 0, 10 5, 0 -5, 0 0))'),
+select 'PG4', ST_AsText(ST_Normalize(ST_AsMVTGeom(
+    ST_GeomFromText('POLYGON ((0 0, 10 0, 10 5, 0 -5, 0 0))'),
 	ST_MakeBox2D(ST_Point(0, 0), ST_Point(4096, 4096)),
-	4096, 0, false));
+    4096, 0, false)));
 select 'PG5', ST_AsText(ST_AsMVTGeom(
 	ST_GeomFromText('POLYGON ((0 0, 10 0, 10 5, 0 -5, 0 0))'),
 	ST_MakeBox2D(ST_Point(0, 0), ST_Point(4096*4096, 4096*4096)),
@@ -34,10 +34,10 @@ SELECT 'PG7', ST_Area(COALESCE(ST_AsMVTGeom(
 	ST_MakeBox2D(ST_Point(-20037508.34, -20037508.34), ST_Point(20037508.34, 20037508.34)),
 	4096, 10, true), 'POLYGON EMPTY'::geometry)) <= 2;
 
-select 'PG8', ST_AsText(ST_AsMVTGeom(
-	ST_GeomFromText('GEOMETRYCOLLECTION(MULTIPOLYGON (((0 0, 10 0, 10 5, 0 -5, 0 0))))'),
+select 'PG8', ST_AsText(ST_Normalize(ST_AsMVTGeom(
+    ST_GeomFromText('GEOMETRYCOLLECTION(MULTIPOLYGON (((0 0, 10 0, 10 5, 0 -5, 0 0))))'),
 	ST_MakeBox2D(ST_Point(0, 0), ST_Point(4096, 4096)),
-	4096, 0, false));
+    4096, 0, false)));
 select 'PG9', ST_Area(ST_AsMVTGeom(
 	ST_GeomFromText('POLYGON ((0 0, 10 0, 10 10, 0 10, 0 0))'),
 	ST_MakeBox2D(ST_Point(0, 0), ST_Point(5, 5)),
@@ -267,10 +267,10 @@ SELECT 'PG43 - ON ', ST_AsText(ST_AsMVTGeom(
 	ST_MakeBox2D(ST_Point(0, 0), ST_Point(100, 100)),
 	10, 0, true));
 
-SELECT 'PG43 - OFF', ST_AsText(ST_AsMVTGeom(
-	ST_GeomFromText('POLYGON((-10 -10, 110 110, -10 110, 110 -10, -10 -10))'),
+SELECT 'PG43 - OFF', ST_AsText(ST_Normalize(ST_AsMVTGeom(
+    ST_GeomFromText('POLYGON((-10 -10, 110 110, -10 110, 110 -10, -10 -10))'),
 	ST_MakeBox2D(ST_Point(0, 0), ST_Point(100, 100)),
-	10, 0, false));
+    10, 0, false)));
 
 -- Geometry type change
 SELECT 'PG44', ST_AsEWKT(ST_AsMVTGeom(
@@ -358,13 +358,13 @@ SELECT 'PG56', ST_AsText(ST_AsMVTGeom(
 -- Different round behaviour between geos and wagyu
 WITH geometry AS
 (
-    SELECT ST_AsText(ST_AsMVTGeom(
-	ST_GeomFromText('POLYGON((0 0, 0 99, 1 101, 100 100, 100 0, 0 0))'),
+SELECT ST_AsText(ST_Normalize(ST_AsMVTGeom(
+    ST_GeomFromText('POLYGON((0 0, 0 99, 1 101, 100 100, 100 0, 0 0))'),
 	ST_MakeBox2D(ST_Point(0, 0), ST_Point(100, 100)),
-	100, 0, true)) as g
+    100, 0, true))) as g
 )
 SELECT 'PG57',
-        g = 'POLYGON((100 0,100 100,0 100,0 1,1 0,100 0))' OR g = 'POLYGON((0 1,0 0,100 0,100 100,0 100,0 1))'
+       g = '' POLYGON((0 1,0 100,100 100,100 0,1 0,0 1))'' OR g = ''POLYGON((0 0,0 1,0 100,100 100,100 0,0 0))''
 FROM geometry;
 
 -- Geometrycollection test
