diff --git a/regress/operators.sql b/regress/operators.sql
index 206fd88dd..9c845a363 100644
--- a/regress/operators.sql
+++ b/regress/operators.sql
@@ -79,8 +79,8 @@ select 'ab3',ST_MakeEnvelope(2,5,4,8) |>> ST_MakeEnvelope(2,2,4,4); --t
 
 -- same as           =
 
-select 'eq1',ST_MakeEnvelope(2,2,4,4) = ST_MakeEnvelope(2,2,4,4); -- f
-select 'eq2',ST_MakeEnvelope(2,4,4,8) = 'LINESTRING(2 4,4 8)'::geometry; -- t
+select 'eq1',ST_MakeEnvelope(2,2,4,4) = ST_MakeEnvelope(2,2,4,4); -- t
+select 'eq2',ST_MakeEnvelope(2,4,4,8) = 'LINESTRING(2 4,4 8)'::geometry; -- f
 select 'eq3',ST_MakePoint(0,0) = ST_MakePoint(1,0); -- f
 
 -- box centroid distance  <->
@@ -141,7 +141,7 @@ SELECT 'ndovm2', array_agg(i) FROM v WHERE g &&& 'POINTZ(0 0 1)'::geometry
 ORDER BY 1;
 
 -- GROUP BY on empty
-SELECT '#3777', ST_AsText(geom), count(*)
+CREATE TABLE t1 AS SELECT ST_AsText(geom) as geom
 FROM (VALUES
     ('POINT(0 0)'::geometry),
     ('POINT(0 0)'::geometry),
@@ -150,10 +150,13 @@ FROM (VALUES
     ('LINESTRING(0 0,0 1)'::geometry),
     ('GEOMETRYCOLLECTION EMPTY'::geometry),
     ('POINT EMPTY'::geometry)
-) AS f(geom)
+) AS f(geom) DISTRIBUTED BY (geom);
+
+SELECT '#3777', ST_AsText(geom), count(*)
+FROM t1
 GROUP BY geom ORDER BY 2;
 
-SELECT '#3777.1', ST_AsText(geom), count(*)
+CREATE TABLE t2 AS SELECT ST_AsText(geom) as geom
 FROM (VALUES
     ('POINT(0 0)'::geometry),
     ('POINT(0 0)'::geometry),
@@ -162,5 +165,10 @@ FROM (VALUES
     ('POINT(0 1)'::geometry),
     ('LINESTRING(0 0,0 1)'::geometry),
     ('GEOMETRYCOLLECTION EMPTY'::geometry)
-) AS f(geom)
+) AS f(geom) DISTRIBUTED BY (geom);
+
+SELECT '#3777.1', ST_AsText(geom), count(*)
+FROM t2
 GROUP BY geom ORDER BY 2;
+
+DROP TABLE IF EXISTS t1, t2;
