diff --git a/regress/run_test.pl b/regress/run_test.pl
index 65886753f..232ad1ba1 100755
--- a/regress/run_test.pl
+++ b/regress/run_test.pl
@@ -7,6 +7,7 @@
 # Copyright (C) 2012-2014 Sandro Santilli <strk@kbt.io>
 # Copyright (C) 2014-2015 Regina Obe <lr@pcorp.us>
 # Copyright (C) 2012-2013 Paul Ramsey <pramsey@cleverelephant.ca>
+# Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 #
 # This is free software; you can redistribute and/or modify it under
 # the terms of the GNU General Public Licence. See the COPYING file.
@@ -49,7 +50,7 @@ our $REGDIR = abs_path(dirname($0));
 our $SHP2PGSQL = $REGDIR . "/../loader/shp2pgsql";
 our $PGSQL2SHP = $REGDIR . "/../loader/pgsql2shp";
 our $RASTER2PGSQL = $REGDIR . "/../raster/loader/raster2pgsql";
-our $sysdiff = !system("diff --strip-trailing-cr $0 $0 2> /dev/null");
+our $sysdiff = !system("/usr/local/greenplum-db/lib/postgresql/pgxs/src/test/regress/gpdiff.pl -b -w -gpd_init " . $REGDIR . "/global_init_file --strip-trailing-cr $0 $0 2> /dev/null");
 
 ##################################################################
 # Parse command line opts
@@ -745,6 +746,7 @@ sub run_simple_test
 	@lines = grep(!/^(LOG|SET|TRUNCATE|DISCARD)/, @lines);
 	@lines = grep(!/^LINE \d/, @lines);
 	@lines = grep(!/^\s+$/, @lines);
+    @lines = grep(!/NOTICE:  Table doesn't have 'distributed by' clause/i, @lines);
 
 	# Morph values into expected forms
 	for ( my $i = 0; $i < @lines; $i++ )
@@ -1216,7 +1218,8 @@ sub run_raster_loader_test
 sub count_db_objects
 {
 	my $count = sql("WITH counts as (
-		select count(*) from pg_type union all
+		select count(*) from pg_type
+            where typname NOT LIKE '__%' union all
 		select count(*) from pg_proc union all
 		select count(*) from pg_cast union all
 		select count(*) from pg_aggregate union all
@@ -1247,8 +1250,6 @@ sub create_spatial
 
   $rv = create_db();
 
-	$cmd = "createlang plpgsql $DB >> $REGRESS_LOG 2>&1";
-	$rv = system($cmd);
 
 	# Count database objects before installing anything
 	$OBJ_COUNT_PRE = count_db_objects();
@@ -1675,7 +1676,10 @@ sub diff
 	my $diffstr = '';
 
 	if ( $sysdiff ) {
-		$diffstr = `diff --strip-trailing-cr -u $expected_file $obtained_file 2>&1`;
+
+		#$diffstr = `diff --strip-trailing-cr -u $expected_file $obtained_file 2>&1`;
+		$diffstr = `/usr/local/greenplum-db/lib/postgresql/pgxs/src/test/regress/gpdiff.pl -b -w -gpd_init $REGDIR/global_init_file --strip-trailing-cr $expected_file $obtained_file 2>&1`;
+
 		return $diffstr;
 	}
 
