diff --git a/regress/swapordinates_expected b/regress/swapordinates_expected
index 57ecf1563..64ef44e63 100644
--- a/regress/swapordinates_expected
+++ b/regress/swapordinates_expected
@@ -1,9 +1,9 @@
 flip1|POINT(1 0)
 flip2|GEOMETRYCOLLECTION(POINT(2 1),MULTIPOLYGON(((1 0,2 1,1 2,1 0)),((0 10,0 20,10 20,0 10),(2 2,2 4,4 4,2 2))),LINESTRING(1 0,0 1))
-ERROR:  Geometry does not have an M ordinate
-ERROR:  Geometry does not have a Z ordinate
-ERROR:  Invalid ordinate specification. Need two letters from the set (x,y,z,m). Got 'x'
-ERROR:  Invalid ordinate name 'p'. Expected x,y,z or m
+ERROR:  Geometry does not have an M ordinate (lwgeom_pg.c:128)
+ERROR:  Geometry does not have a Z ordinate (lwgeom_pg.c:128)
+ERROR:  Invalid ordinate specification. Need two letters from the set (x,y,z,m). Got 'x' (lwgeom_pg.c:128)
+ERROR:  Invalid ordinate name 'p'. Expected x,y,z or m (lwgeom_pg.c:128)
 swap1|POINT Z (2 1 0)
 swap2|POINT M (0 2 1)
 swap3|POINT ZM (0 1 3 2)
