diff --git a/raster/scripts/python/rtgdalraster.py b/raster/scripts/python/rtgdalraster.py
index 2b32c70f6..427f06516 100755
--- a/raster/scripts/python/rtgdalraster.py
+++ b/raster/scripts/python/rtgdalraster.py
@@ -13,6 +13,7 @@
 # Copyright (C) 2011 Regents of the University of California
 #   <bkpark@ucdavis.edu>
 # Copyright (C) 2009 Mateusz Loskot <mateusz@loskot.net>
+# Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 # 
 # This program is free software; you can redistribute it and/or
 # modify it under the terms of the GNU General Public License
