diff --git a/raster/scripts/python/rtreader.py b/raster/scripts/python/rtreader.py
index 4403be577..187565a93 100755
--- a/raster/scripts/python/rtreader.py
+++ b/raster/scripts/python/rtreader.py
@@ -4,6 +4,7 @@
 # A simple driver to read RASTER field data directly from PostGIS/WKTRaster.
 #
 # Copyright (C) 2009 Mateusz Loskot <mateusz@loskot.net>
+# Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 # 
 # This program is free software; you can redistribute it and/or
 # modify it under the terms of the GNU General Public License
