diff --git a/macros/iconv.m4 b/macros/iconv.m4
index 66bc76f48..00ca0fe6f 100644
--- a/macros/iconv.m4
+++ b/macros/iconv.m4
@@ -1,5 +1,6 @@
 # iconv.m4 serial AM6 (gettext-0.17)
 dnl Copyright (C) 2000-2002, 2007 Free Software Foundation, Inc.
+dnl Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 dnl This file is free software; the Free Software Foundation
 dnl gives unlimited permission to copy and/or distribute it,
 dnl with or without modifications, as long as this notice is preserved.
