diff --git a/macros/intldir.m4 b/macros/intldir.m4
index 7a28843f6..55957cba8 100644
--- a/macros/intldir.m4
+++ b/macros/intldir.m4
@@ -1,5 +1,6 @@
 # intldir.m4 serial 1 (gettext-0.16)
 dnl Copyright (C) 2006 Free Software Foundation, Inc.
+dnl Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 dnl This file is free software; the Free Software Foundation
 dnl gives unlimited permission to copy and/or distribute it,
 dnl with or without modifications, as long as this notice is preserved.
