diff --git a/macros/intl.m4 b/macros/intl.m4
index 934408bb9..dfcc7f450 100644
--- a/macros/intl.m4
+++ b/macros/intl.m4
@@ -1,5 +1,6 @@
 # intl.m4 serial 8 (gettext-0.17)
 dnl Copyright (C) 1995-2007 Free Software Foundation, Inc.
+dnl Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 dnl This file is free software; the Free Software Foundation
 dnl gives unlimited permission to copy and/or distribute it,
 dnl with or without modifications, as long as this notice is preserved.
