diff --git a/macros/lib-prefix.m4 b/macros/lib-prefix.m4
index a8684e17e..863695dc3 100644
--- a/macros/lib-prefix.m4
+++ b/macros/lib-prefix.m4
@@ -1,5 +1,6 @@
 # lib-prefix.m4 serial 5 (gettext-0.15)
 dnl Copyright (C) 2001-2005 Free Software Foundation, Inc.
+dnl Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 dnl This file is free software; the Free Software Foundation
 dnl gives unlimited permission to copy and/or distribute it,
 dnl with or without modifications, as long as this notice is preserved.
