diff --git a/macros/pkg.m4 b/macros/pkg.m4
index 4688002e4..275f50ed3 100644
--- a/macros/pkg.m4
+++ b/macros/pkg.m4
@@ -3,6 +3,7 @@ dnl serial 11 (pkg-config-0.29)
 dnl
 dnl Copyright © 2004 Scott James Remnant <scott@netsplit.com>.
 dnl Copyright © 2012-2015 Dan Nicholson <dbn.lists@gmail.com>
+dnl Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 dnl
 dnl This program is free software; you can redistribute it and/or modify
 dnl it under the terms of the GNU General Public License as published by
