diff --git a/topology/Makefile.in b/topology/Makefile.in
index 4f89fbaef..ffb3bd530 100644
--- a/topology/Makefile.in
+++ b/topology/Makefile.in
@@ -6,6 +6,7 @@
 # * Copyright (C) 2010-2011 Sandro Santilli <strk@kbt.io>
 # * Copyright (C) 2008 Mark Cave-Ayland
 # * Copyright (C) 2005 Refractions Research Inc.
+# * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 # *
 # * This is free software; you can redistribute and/or modify it under
 # * the terms of the GNU General Public Licence. See the COPYING file.
