diff --git a/raster/scripts/plpgsql/st_clip.sql b/raster/scripts/plpgsql/st_clip.sql
index 726649650..cf32ffb65 100644
--- a/raster/scripts/plpgsql/st_clip.sql
+++ b/raster/scripts/plpgsql/st_clip.sql
@@ -2,6 +2,7 @@
 --
 --
 -- Copyright (c) 2009-2010 Pierre Racine <pierre.racine@sbf.ulaval.ca>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 ----------------------------------------------------------------------
 
