diff --git a/raster/scripts/plpgsql/st_deleteband.sql b/raster/scripts/plpgsql/st_deleteband.sql
index aef0f7652..2041531e7 100644
--- a/raster/scripts/plpgsql/st_deleteband.sql
+++ b/raster/scripts/plpgsql/st_deleteband.sql
@@ -2,6 +2,7 @@
 --
 --
 -- Copyright (c) 2009-2010 Pierre Racine <pierre.racine@sbf.ulaval.ca>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 ----------------------------------------------------------------------
 
