diff --git a/raster/scripts/plpgsql/st_setarray.sql b/raster/scripts/plpgsql/st_setarray.sql
index a808a1656..06e5a99a8 100644
--- a/raster/scripts/plpgsql/st_setarray.sql
+++ b/raster/scripts/plpgsql/st_setarray.sql
@@ -1,5 +1,6 @@
 ----------------------------------------------------------------------
 -- Copyright (c) 2009-2010 Pierre Racine <pierre.racine@sbf.ulaval.ca>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 ----------------------------------------------------------------------
 
 CREATE OR REPLACE FUNCTION ST_SetArea(rast raster, band, x int, y int, width int, height int, val float8)
