diff --git a/raster/scripts/plpgsql/st_union.sql b/raster/scripts/plpgsql/st_union.sql
index 18af1d579..d672d4988 100644
--- a/raster/scripts/plpgsql/st_union.sql
+++ b/raster/scripts/plpgsql/st_union.sql
@@ -2,6 +2,7 @@
 --
 --
 -- Copyright (c) 2009-2010 Pierre Racine <pierre.racine@sbf.ulaval.ca>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 ----------------------------------------------------------------------
 
