diff --git a/extensions/postgis_tiger_geocoder/sql_bits/add_search_path.sql.in b/extensions/postgis_tiger_geocoder/sql_bits/add_search_path.sql.in
index e3d3d5833..15f3e1812 100644
--- a/extensions/postgis_tiger_geocoder/sql_bits/add_search_path.sql.in
+++ b/extensions/postgis_tiger_geocoder/sql_bits/add_search_path.sql.in
@@ -5,6 +5,7 @@
 -- http://postgis.net
 --
 -- Copyright (C) 2012 Regina Obe <lr@pcorp.us>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 -- This is free software; you can redistribute and/or modify it under
 -- the terms of the GNU General Public Licence. See the COPYING file.
