diff --git a/loader/shapefil.h b/loader/shapefil.h
index 6eae41672..28f50cc69 100644
--- a/loader/shapefil.h
+++ b/loader/shapefil.h
@@ -9,6 +9,7 @@
  *
  ******************************************************************************
  * Copyright (c) 1999, Frank Warmerdam
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  * This software is available under the following "MIT Style" license,
  * or at the option of the licensee under the LGPL (see LICENSE.LGPL).  This
