diff --git a/libpgcommon/common.h b/libpgcommon/common.h
index a84b70c63..22f75e6e5 100644
--- a/libpgcommon/common.h
+++ b/libpgcommon/common.h
@@ -3,6 +3,7 @@
  * PostGIS - Spatial Types for PostgreSQL
  * http://postgis.net
  * Copyright 2011 OSGeo
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  * This is free software; you can redistribute and/or modify it under
  * the terms of the GNU General Public Licence. See the COPYING file.
