diff --git a/topology/sql/populate.sql.in b/topology/sql/populate.sql.in
index 7974182af..3f247c762 100644
--- a/topology/sql/populate.sql.in
+++ b/topology/sql/populate.sql.in
@@ -4,6 +4,7 @@
 -- http://postgis.net
 --
 -- Copyright (C) 2010-2012 Sandro Santilli <strk@kbt.io>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 -- This is free software; you can redistribute and/or modify it under
 -- the terms of the GNU General Public Licence. See the COPYING file.
