diff --git a/liblwgeom/cunit/cu_chaikin.c b/liblwgeom/cunit/cu_chaikin.c
index 60b398b3e..89eb9e75a 100644
--- a/liblwgeom/cunit/cu_chaikin.c
+++ b/liblwgeom/cunit/cu_chaikin.c
@@ -4,6 +4,7 @@
  * http://postgis.net
  *
  * Copyright 2018 Nicklas Avén <nicklas.aven@jordogskog.no>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  * This is free software; you can redistribute and/or modify it under
  * the terms of the GNU General Public Licence. See the COPYING file.
