diff --git a/liblwgeom/cunit/cu_geos.c b/liblwgeom/cunit/cu_geos.c
index ec3cedc0b..164fe362e 100644
--- a/liblwgeom/cunit/cu_geos.c
+++ b/liblwgeom/cunit/cu_geos.c
@@ -4,6 +4,7 @@
  * http://postgis.net
  *
  * Copyright 2011 Sandro Santilli <strk@kbt.io>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  * This is free software; you can redistribute and/or modify it under
  * the terms of the GNU General Public Licence. See the COPYING file.
@@ -126,19 +127,29 @@ test_geos_offsetcurve_crash(void)
 static void
 test_geos_makevalid(void)
 {
-	uint8_t* wkb;
-	char* out_ewkt;
-	LWGEOM* geom1;
-	LWGEOM* geom2;
-
-	wkb = (uint8_t*) "\001\003\000\000\000\001\000\000\000\011\000\000\000b\020X9 }\366@7\211A\340\235I\034A\316\326t18}\366@\306g\347\323\230I\034Ay\351&18}\366@\331\316\367\323\230I\034A\372~j\274\370}\366@\315\314\314LpI\034A\343\245\233\304R}\366@R\270\036\005?I\034A\315\314\314\314Z~\366@\343\245\233\304\007I\034A\004V\016-\242}\366@\252\361\322M\323H\034A\351&1\010\306{\366@H\341z\0247I\034Ab\020X9 }\366@7\211A\340\235I\034A";
-	geom1 = lwgeom_from_wkb(wkb, 157, LW_PARSER_CHECK_NONE);
-	geom2 = lwgeom_make_valid(geom1);
-	out_ewkt = lwgeom_to_ewkt((LWGEOM*)geom2);
-	ASSERT_STRING_EQUAL(out_ewkt, "GEOMETRYCOLLECTION(POLYGON((92114.014 463463.469,92115.5120743 463462.206937,92115.512 463462.207,92127.546 463452.075,92117.173 463439.755,92133.675 463425.942,92122.136 463412.826,92092.377 463437.77,92114.014 463463.469)),MULTIPOINT(92115.5120743 463462.206937,92122.136 463412.826))");
-	lwfree(out_ewkt);
-	lwgeom_free(geom1);
-	lwgeom_free(geom2);
+    uint8_t* wkb;
+    char* out_ewkt;
+    LWGEOM* geom1;
+    LWGEOM* geom2;
+    LWGEOM* geom3;
+    wkb = (uint8_t*) "\001\003\000\000\000\001\000\000\000\011\000\000\000b\020X9 }\366@7\211A\340\235I\034A\316\326t18}\366@\306g\347\323\230I\034Ay\351&18}\366@\331\316\367\323\230I\034A\372~j\274\370}\366@\315\314\314LpI\034A\343\245\233\304R}\366@R\270\036\005?I\034A\315\314\314\314Z~\366@\343\245\233\304\007I\034A\004V\016-\242}\366@\252\361\322M\323H\034A\351&1\010\306{\366@H\341z\0247I\034Ab\020X9 }\366@7\211A\340\235I\034A";
+    geom1 = lwgeom_from_wkb(wkb, 157, LW_PARSER_CHECK_NONE);
+    geom2 = lwgeom_make_valid(geom1);
+    geom3 = lwgeom_normalize(geom2); //so GEOS 3.9 and 3.10 agree
+    out_ewkt = lwgeom_to_ewkt((LWGEOM*)geom3);
+#if POSTGIS_GEOS_VERSION < 39
+    ASSERT_STRING_EQUAL(
+            out_ewkt,
+            "GEOMETRYCOLLECTION(POLYGON((92092.377 463437.77,92114.014 463463.469,92115.5120743 463462.206937,92115.512 463462.207,92127.546 463452.075,92117.173 463439.755,92133.675 463425.942,92122.136 463412.826,92092.377 463437.77)),MULTIPOINT(92122.136 463412.826,92115.5120743 463462.206937))");
+#else
+    ASSERT_STRING_EQUAL(
+            out_ewkt,
+            "POLYGON((92092.377 463437.77,92114.014 463463.469,92115.512 463462.207,92115.5120743 463462.206937,92127.546 463452.075,92117.173 463439.755,92133.675 463425.942,92122.136 463412.826,92092.377 463437.77))");
+#endif
+    lwfree(out_ewkt);
+    lwgeom_free(geom1);
+    lwgeom_free(geom2);
+    lwgeom_free(geom3);
 }
 
 
