diff --git a/raster/Makefile.in b/raster/Makefile.in
index fa6e1048b..2fa2a3a94 100644
--- a/raster/Makefile.in
+++ b/raster/Makefile.in
@@ -3,6 +3,7 @@
 # Master Makefile used to build WKT Raster
 #
 # Copyright (c) 2009 Sandro Santilli <strk@kbt.io>
+# Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 #
 # This program is free software; you can redistribute it and/or
 # modify it under the terms of the GNU General Public License
