diff --git a/liblwgeom/cunit/cu_bytebuffer.c b/liblwgeom/cunit/cu_bytebuffer.c
index 64dc80a6e..3a48e47c3 100644
--- a/liblwgeom/cunit/cu_bytebuffer.c
+++ b/liblwgeom/cunit/cu_bytebuffer.c
@@ -5,6 +5,7 @@
  * http://postgis.net
  *
  * Copyright 2012 Sandro Santilli <strk@kbt.io>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  * This is free software; you can redistribute and/or modify it under
  * the terms of the GNU General Public Licence. See the COPYING file.
