diff --git a/liblwgeom/cunit/cu_lwstroke.c b/liblwgeom/cunit/cu_lwstroke.c
index ebe93b02a..ebe7f9f70 100644
--- a/liblwgeom/cunit/cu_lwstroke.c
+++ b/liblwgeom/cunit/cu_lwstroke.c
@@ -4,6 +4,7 @@
  * http://postgis.net
  *
  * Copyright (C) 2017 Sandro Santilli <strk@kbt.io>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  * This is free software; you can redistribute and/or modify it under
  * the terms of the GNU General Public Licence. See the COPYING file.
