diff --git a/liblwgeom/cunit/cu_stringbuffer.c b/liblwgeom/cunit/cu_stringbuffer.c
index b6479426a..8ba3384d2 100644
--- a/liblwgeom/cunit/cu_stringbuffer.c
+++ b/liblwgeom/cunit/cu_stringbuffer.c
@@ -4,6 +4,7 @@
  * http://postgis.net
  *
  * Copyright 2012 Sandro Santilli <strk@kbt.io>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  * This is free software; you can redistribute and/or modify it under
  * the terms of the GNU General Public Licence. See the COPYING file.
