diff --git a/extras/tiger_geocoder/tiger_loader_2015.sql b/extras/tiger_geocoder/tiger_loader_2015.sql
index fe0bdf62f..3a722ce0c 100644
--- a/extras/tiger_geocoder/tiger_loader_2015.sql
+++ b/extras/tiger_geocoder/tiger_loader_2015.sql
@@ -3,6 +3,7 @@
 -- http://postgis.net
 --
 -- Copyright (C) 2012-2015 Regina Obe and Leo Hsu
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 -- Paragon Corporation
 --
 -- This is free software; you can redistribute and/or modify it under
