diff --git a/liblwgeom/bytebuffer.h b/liblwgeom/bytebuffer.h
index 8a2a15241..b55ac1de1 100644
--- a/liblwgeom/bytebuffer.h
+++ b/liblwgeom/bytebuffer.h
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2015 Nicklas Avén <nicklas.aven@jordogskog.no>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
