diff --git a/liblwgeom/g_serialized.c b/liblwgeom/g_serialized.c
index 2b5312fb0..56a517c4e 100644
--- a/liblwgeom/g_serialized.c
+++ b/liblwgeom/g_serialized.c
@@ -20,13 +20,14 @@
  *
  * Copyright 2009 Paul Ramsey <pramsey@cleverelephant.ca>
  * Copyright 2017 Darafei Praliaskouski <me@komzpa.net>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
-
 #include "liblwgeom_internal.h"
 #include "lwgeom_log.h"
 #include "lwgeodetic.h"
+#include "postgres.h" //for VARSIZE and SET_VARSIZE
 
 /***********************************************************************
 * GSERIALIZED metadata utility functions.
@@ -143,8 +144,8 @@ GSERIALIZED* gserialized_copy(const GSERIALIZED *g)
 {
 	GSERIALIZED *g_out = NULL;
 	assert(g);
-	g_out = (GSERIALIZED*)lwalloc(SIZE_GET(g->size));
-	memcpy((uint8_t*)g_out,(uint8_t*)g,SIZE_GET(g->size));
+	g_out = (GSERIALIZED*)lwalloc(VARSIZE(g));
+	memcpy((uint8_t*)g_out,(uint8_t*)g,VARSIZE(g));
 	return g_out;
 }
 
@@ -296,8 +297,8 @@ int gserialized_cmp(const GSERIALIZED *g1, const GSERIALIZED *g2)
 	int g1_is_empty, g2_is_empty, cmp;
 	GBOX box1, box2;
 	uint64_t hash1, hash2;
-	size_t sz1 = SIZE_GET(g1->size);
-	size_t sz2 = SIZE_GET(g2->size);
+	size_t sz1 = VARSIZE(g1);
+	size_t sz2 = VARSIZE(g2);
 	union floatuint x, y;
 
 	/*
@@ -1231,7 +1232,8 @@ GSERIALIZED* gserialized_from_lwgeom(LWGEOM *geom, size_t *size)
 	** We are aping PgSQL code here, PostGIS code should use
 	** VARSIZE to set this for real.
 	*/
-	g->size = return_size << 2;
+	//g->size = return_size << 2;
+	SET_VARSIZE(g, return_size);
 
 	/* Set the SRID! */
 	gserialized_set_srid(g, geom->srid);
