diff --git a/liblwgeom/lwgeom_wrapx.c b/liblwgeom/lwgeom_wrapx.c
index 0f50ae2b6..7c380a374 100644
--- a/liblwgeom/lwgeom_wrapx.c
+++ b/liblwgeom/lwgeom_wrapx.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2016 Sandro Santilli <strk@kbt.io>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
