diff --git a/liblwgeom/lwgeom_api.c b/liblwgeom/lwgeom_api.c
index 64189f596..1b6670976 100644
--- a/liblwgeom/lwgeom_api.c
+++ b/liblwgeom/lwgeom_api.c
@@ -20,6 +20,7 @@
  *
  * Copyright 2001-2006 Refractions Research Inc.
  * Copyright 2017 Darafei Praliaskouski <me@komzpa.net>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
