diff --git a/liblwgeom/lwgeom_median.c b/liblwgeom/lwgeom_median.c
index 03d37b12c..de18f45fd 100644
--- a/liblwgeom/lwgeom_median.c
+++ b/liblwgeom/lwgeom_median.c
@@ -20,6 +20,7 @@
  *
  * Copyright 2015 Daniel Baston <dbaston@gmail.com>
  * Copyright 2017 Darafei Praliaskouski <me@komzpa.net>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
