diff --git a/liblwgeom/lwgeom_geos_node.c b/liblwgeom/lwgeom_geos_node.c
index 8e5e8c3bc..9b6412135 100644
--- a/liblwgeom/lwgeom_geos_node.c
+++ b/liblwgeom/lwgeom_geos_node.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright (C) 2011 Sandro Santilli <strk@kbt.io>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
