diff --git a/liblwgeom/lwin_twkb.c b/liblwgeom/lwin_twkb.c
index ed838035f..88cc6e083 100644
--- a/liblwgeom/lwin_twkb.c
+++ b/liblwgeom/lwin_twkb.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright (C) 2014 Nicklas Avén
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
