diff --git a/liblwgeom/lwin_encoded_polyline.c b/liblwgeom/lwin_encoded_polyline.c
index 9d486f576..93695095d 100644
--- a/liblwgeom/lwin_encoded_polyline.c
+++ b/liblwgeom/lwin_encoded_polyline.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
 * Copyright 2014 Kashif Rasul <kashif.rasul@gmail.com> and
+* Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
