diff --git a/liblwgeom/lwline.c b/liblwgeom/lwline.c
index 0c7fa8e64..fe7728ebd 100644
--- a/liblwgeom/lwline.c
+++ b/liblwgeom/lwline.c
@@ -20,6 +20,7 @@
  *
  * Copyright (C) 2012 Sandro Santilli <strk@kbt.io>
  * Copyright (C) 2001-2006 Refractions Research Inc.
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
