diff --git a/liblwgeom/lwout_geojson.c b/liblwgeom/lwout_geojson.c
index 90a23787b..7c45736ad 100644
--- a/liblwgeom/lwout_geojson.c
+++ b/liblwgeom/lwout_geojson.c
@@ -20,6 +20,7 @@
  *
  * Copyright 2001-2003 Refractions Research Inc.
  * Copyright 2009-2010 Olivier Courtin <olivier.courtin@oslandia.com>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
