diff --git a/liblwgeom/lwout_kml.c b/liblwgeom/lwout_kml.c
index fa8912df1..d554435e4 100644
--- a/liblwgeom/lwout_kml.c
+++ b/liblwgeom/lwout_kml.c
@@ -20,6 +20,7 @@
  *
  * Copyright 2006 Corporacion Autonoma Regional de Santander
  * Copyright 2010 Paul Ramsey <pramsey@cleverelephant.ca>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
