diff --git a/liblwgeom/lwtree.c b/liblwgeom/lwtree.c
index c3b3f1215..120a6e10a 100644
--- a/liblwgeom/lwtree.c
+++ b/liblwgeom/lwtree.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright (C) 2009-2012 Paul Ramsey <pramsey@cleverelephant.ca>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
