diff --git a/liblwgeom/measures.c b/liblwgeom/measures.c
index 3793dc3b7..e0c2e9ae4 100644
--- a/liblwgeom/measures.c
+++ b/liblwgeom/measures.c
@@ -21,6 +21,7 @@
  * Copyright 2001-2006 Refractions Research Inc.
  * Copyright 2010 Nicklas Avén
  * Copyright 2012 Paul Ramsey
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
