diff --git a/libpgcommon/cunit/Makefile.in b/libpgcommon/cunit/Makefile.in
index b4a660ba6..1c68b5213 100644
--- a/libpgcommon/cunit/Makefile.in
+++ b/libpgcommon/cunit/Makefile.in
@@ -3,6 +3,7 @@
 # * PostGIS - Spatial Types for PostgreSQL
 # * http://postgis.net
 # * Copyright 2008 Mark Cave-Ayland
+# * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 # *
 # * This is free software; you can redistribute and/or modify it under
 # * the terms of the GNU General Public Licence. See the COPYING file.
