diff --git a/raster/loader/raster2pgsql.h b/raster/loader/raster2pgsql.h
index 7e8a574e9..92c5c843b 100644
--- a/raster/loader/raster2pgsql.h
+++ b/raster/loader/raster2pgsql.h
@@ -8,6 +8,7 @@
  * Copyright 2009 Mark Cave-Ayland <mark.cave-ayland@siriusit.co.uk>
  * Copyright (C) 2011 Regents of the University of California
  *   <bkpark@ucdavis.edu>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License
