diff --git a/raster/scripts/python/ovdump.py b/raster/scripts/python/ovdump.py
index f8e6d79b2..8705a1624 100755
--- a/raster/scripts/python/ovdump.py
+++ b/raster/scripts/python/ovdump.py
@@ -8,6 +8,7 @@
 #       which dumps overview as complete geospatial raster dataset.
 #
 # Copyright (C) 2009 Mateusz Loskot <mateusz@loskot.net>
+# Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 # 
 # This program is free software; you can redistribute it and/or
 # modify it under the terms of the GNU General Public License
