diff --git a/raster/scripts/python/raster2pgsql.py b/raster/scripts/python/raster2pgsql.py
index 677eff49e..f0689a4fd 100755
--- a/raster/scripts/python/raster2pgsql.py
+++ b/raster/scripts/python/raster2pgsql.py
@@ -14,6 +14,7 @@
 # Copyright (C) 2009-2010 Mateusz Loskot <mateusz@loskot.net>
 # Copyright (C) 2009-2011 Pierre Racine <pierre.racine@sbf.ulaval.ca>
 # Copyright (C) 2009-2010 Jorge Arevalo <jorge.arevalo@deimos-space.com>
+# Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 # 
 # This program is free software; you can redistribute it and/or
 # modify it under the terms of the GNU General Public License
