diff --git a/raster/scripts/python/rtrowdump.py b/raster/scripts/python/rtrowdump.py
index 451899395..ec9f7bd74 100755
--- a/raster/scripts/python/rtrowdump.py
+++ b/raster/scripts/python/rtrowdump.py
@@ -10,6 +10,7 @@
 #
 ###############################################################################
 # Copyright (C) 2009 Mateusz Loskot <mateusz@loskot.net>
+# Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 # 
 # This program is free software; you can redistribute it and/or
 # modify it under the terms of the GNU General Public License
