diff --git a/raster/scripts/python/window.py b/raster/scripts/python/window.py
index fa78e4a5c..f99675118 100755
--- a/raster/scripts/python/window.py
+++ b/raster/scripts/python/window.py
@@ -6,6 +6,7 @@
 #
 ##############################################################################
 # Copyright (C) 2009 Mateusz Loskot <mateusz@loskot.net>
+# Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 # 
 # This program is free software; you can redistribute it and/or
 # modify it under the terms of the GNU General Public License
