diff --git a/raster/test/regress/bug_test_car5.sql b/raster/test/regress/bug_test_car5.sql
index 3e2ece7dd..40baefd6c 100644
--- a/raster/test/regress/bug_test_car5.sql
+++ b/raster/test/regress/bug_test_car5.sql
@@ -1,5 +1,6 @@
 --
 -- Copyright (c) 2009 Mateusz Loskot <mateusz@loskot.net>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 -- This program is free software; you can redistribute it and/or
 -- modify it under the terms of the GNU General Public License
