diff --git a/raster/test/regress/load_outdb.sql b/raster/test/regress/load_outdb.sql
index 0800b091b..d0c51feec 100644
--- a/raster/test/regress/load_outdb.sql
+++ b/raster/test/regress/load_outdb.sql
@@ -1,4 +1,5 @@
 SET postgis.enable_outdb_rasters = false;
+SET client_min_messages = ERROR;
 SELECT count(*) FROM raster_outdb_template;
 SELECT DISTINCT ST_BandFileSize(rast) FROM raster_outdb_template;
 SELECT DISTINCT ST_BandFileTimestamp(rast) FROM raster_outdb_template;
@@ -16,3 +17,4 @@ SELECT ST_BandFileTimestamp(rast,10) FROM raster_outdb_template WHERE rid = 1;
 -- valid raster, but file does not exist
 SELECT ST_BandFileSize('0100000100000000000000F03F000000000000F0BF0000000000000000000000000000000000000000000000000000000000000000000000005A003200840000DEADBEEF00'::raster);
 SELECT ST_BandFileTimestamp('0100000100000000000000F03F000000000000F0BF0000000000000000000000000000000000000000000000000000000000000000000000005A003200840000DEADBEEF00'::raster);
+RESET client_min_messages;
