diff --git a/raster/test/regress/Makefile.in b/raster/test/regress/Makefile.in
index ab7049810..2f66715be 100644
--- a/raster/test/regress/Makefile.in
+++ b/raster/test/regress/Makefile.in
@@ -4,6 +4,7 @@
 # Copyright (c) 2011 Jorge Arevalo <jorge.arevalo@deimos-space.com>
 # Copyright (c) 2011-2013 Regents of the University of California
 #   <bkpark@ucdavis.edu>
+# Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 #
 # This is free software; you can redistribute and/or modify it under
 # the terms of the GNU General Public Licence. See the COPYING file.
@@ -140,7 +141,7 @@ TEST_SREL = \
 	rt_samealignment \
 	rt_geos_relationships \
 	rt_iscoveragetile
-	
+
 TEST_BUGS = \
 	bug_test_car5 \
 	permitted_gdal_drivers \
@@ -177,7 +178,7 @@ rtpostgis.sql: ../../rt_pg/rtpostgis.sql
 	$(PERL) -lpe "s'\\\$$libdir'$(REGRESS_INSTALLDIR)/lib'g" $< > $@
 
 
-check: 
+check:
 	$(MAKE) -C ../../../regress staged-install
 	$(PERL) ../../../regress/run_test.pl --raster $(RUNTESTFLAGS) $(TESTS)
 	$(PERL) ../../../regress/run_test.pl --upgrade --raster $(RUNTESTFLAGS) $(TESTS)
