diff --git a/raster/test/regress/rt_addband.sql b/raster/test/regress/rt_addband.sql
index 566b6f8de..2a309b05f 100644
--- a/raster/test/regress/rt_addband.sql
+++ b/raster/test/regress/rt_addband.sql
@@ -1,6 +1,7 @@
 -----------------------------------------------------------------------
 --
 -- Copyright (c) 2010 Pierre Racine <pierre.racine@sbf.ulaval.ca>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 -- This program is free software; you can redistribute it and/or
 -- modify it under the terms of the GNU General Public License
@@ -225,34 +226,42 @@ FROM ST_BandMetadata((SELECT rast FROM raster_outdb_template WHERE rid = 4), ARR
 -- #3020
 SET postgis.gdal_enabled_drivers = 'GTiff';
 SET postgis.enable_outdb_rasters = true;
-WITH foo AS (
+
+CREATE TABLE foo1 AS
 	SELECT
 			path
 	FROM ST_BandMetadata(
 		(SELECT rast FROM raster_outdb_template WHERE rid = 1),
 	 	ARRAY[]::int[]
 	)
-	LIMIT 1
-), raster as (
-SELECT
-	ST_AddBand(
-		ST_MakeEmptyRaster(90, 90, 0., 0., 1, -1, 0, 0, 0),
-		1, foo.path, NULL::int[]
-	) AS rast
-FROM foo
-)
+	LIMIT 1 DISTRIBUTED REPLICATED;
+
+SET postgis.gdal_enabled_drivers = 'GTiff';
+SET postgis.enable_outdb_rasters = true;
+CREATE TABLE raster1 AS
+	SELECT
+		ST_AddBand(
+			ST_MakeEmptyRaster(90, 90, 0., 0., 1, -1, 0, 0, 0),
+			1, foo1.path, NULL::int[]
+		) AS rast
+	FROM foo1;
+
+SET postgis.gdal_enabled_drivers = 'GTiff';
+SET postgis.enable_outdb_rasters = true;
 SELECT
 	ST_Value(rast, 1, 1)
-FROM raster
+FROM raster1
 UNION ALL
 SELECT
 	ST_Value(rast, 6, 45)
-FROM raster
+FROM raster1
 UNION ALL
 SELECT
 	ST_Value(rast, 90, 50)
-FROM raster
+FROM raster1
 UNION ALL
 SELECT
 	ST_Value(rast, 100, 100)
-FROM raster
+FROM raster1;
+
+DROP TABLE raster1;
