diff --git a/raster/test/regress/rt_elevation_functions.sql b/raster/test/regress/rt_elevation_functions.sql
index 5190d5239..4faba8c6e 100644
--- a/raster/test/regress/rt_elevation_functions.sql
+++ b/raster/test/regress/rt_elevation_functions.sql
@@ -165,7 +165,7 @@ WHERE rid = 0
 DO $$
 BEGIN
 -- this ONLY works for PostgreSQL version 9.1 or higher
-IF array_to_string(regexp_matches(split_part(version(), ' ', 2), E'([0-9]+)\.([0-9]+)'), '')::int > 90 THEN
+IF array_to_string(regexp_matches(split_part(version(), ' ', 2), E'([0-9]+)\.([0-9]+)'), '')::int > 95 THEN
 
 	INSERT INTO raster_elevation_out (
 		SELECT
