diff --git a/raster/test/regress/rt_grayscale_expected b/raster/test/regress/rt_grayscale_expected
index 5b01d3280..6d1a79fa7 100644
--- a/raster/test/regress/rt_grayscale_expected
+++ b/raster/test/regress/rt_grayscale_expected
@@ -1,10 +1,3 @@
-NOTICE:  table "raster_grayscale_out" does not exist, skipping
-NOTICE:  table "raster_grayscale_in" does not exist, skipping
-WARNING:  Only the first three elements of 'rastbandargset' will be used
-WARNING:  Only the first three elements of 'rastbandargset' will be used
-WARNING:  Only the first three elements of 'rastbandargset' will be used
-WARNING:  Only the first three elements of 'rastbandargset' will be used
-WARNING:  Only the first three elements of 'rastbandargset' will be used
 1|1|1|{{NULL,128},{254,255}}
 1|2|1|{{NULL,128},{254,255}}
 1|3|1|{{NULL,0},{254,254}}
