diff --git a/raster/test/regress/rt_mapalgebra_mask.sql b/raster/test/regress/rt_mapalgebra_mask.sql
index 3e814dc2b..074da3122 100644
--- a/raster/test/regress/rt_mapalgebra_mask.sql
+++ b/raster/test/regress/rt_mapalgebra_mask.sql
@@ -34,8 +34,6 @@ CREATE OR REPLACE FUNCTION raster_nmapalgebra_test(
 	END;
 	$$ LANGUAGE 'plpgsql' IMMUTABLE;
 
-SET client_min_messages TO notice;
-
 select st_mapalgebra(rast,1,'raster_nmapalgebra_test(double precision[], int[], text[])'::regprocedure,ARRAY[[1,1],[1,1],[1,1]]::double precision[],false) from raster_nmapalgebra_mask_in;
 
 select st_mapalgebra(rast,1,'raster_nmapalgebra_test(double precision[], int[], text[])'::regprocedure,ARRAY[[1,1,1],[1,1,1]]::double precision[],false) from raster_nmapalgebra_mask_in;
@@ -121,4 +119,6 @@ FROM (SELECT rid, st_dumpvalues(st_mapalgebra(rast,1,'raster_nmapalgebra_test(do
 ORDER BY rid, (dv).nband;
 
 DROP FUNCTION IF EXISTS raster_nmapalgebra_test(double precision[], int[], text[]);
-DROP TABLE IF EXISTS raster_nmapalgebra_mask_in;
\ No newline at end of file
+DROP TABLE IF EXISTS raster_nmapalgebra_mask_in;
+
+RESET client_min_messages;
\ No newline at end of file
