diff --git a/raster/test/regress/rt_rastertoworldcoord_expected b/raster/test/regress/rt_rastertoworldcoord_expected
index fb3793a9b..1ab014957 100644
--- a/raster/test/regress/rt_rastertoworldcoord_expected
+++ b/raster/test/regress/rt_rastertoworldcoord_expected
@@ -99,10 +99,6 @@
 13|-2
 14|-1
 NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
 0|-2
 1|0
 2|1
@@ -114,10 +110,6 @@ NOTICE:  Pixel row and column required for computing longitude and latitude of a
 13|
 14|
 NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
 0|-2
 1|0
 2|1
@@ -129,10 +121,6 @@ NOTICE:  Pixel row and column required for computing longitude and latitude of a
 13|
 14|
 NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
 0|-1
 1|1
 2|2
@@ -144,10 +132,6 @@ NOTICE:  Pixel row and column required for computing longitude and latitude of a
 13|
 14|
 NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
 0|-3
 1|-1
 2|0
@@ -159,10 +143,6 @@ NOTICE:  Pixel row and column required for computing longitude and latitude of a
 13|
 14|
 NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
 0|-4
 1|-2
 2|-1
@@ -224,10 +204,6 @@ NOTICE:  Pixel row and column required for computing longitude and latitude of a
 13|2
 14|3
 NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
 0|-2
 1|0
 2|-1
@@ -239,10 +215,6 @@ NOTICE:  Pixel row and column required for computing longitude and latitude of a
 13|
 14|
 NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
 0|-2
 1|0
 2|-1
@@ -254,10 +226,6 @@ NOTICE:  Pixel row and column required for computing longitude and latitude of a
 13|
 14|
 NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
 0|-1
 1|1
 2|0
@@ -269,10 +237,6 @@ NOTICE:  Pixel row and column required for computing longitude and latitude of a
 13|
 14|
 NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
 0|-3
 1|-1
 2|-2
@@ -284,10 +248,6 @@ NOTICE:  Pixel row and column required for computing longitude and latitude of a
 13|
 14|
 NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
 0|-4
 1|-2
 2|-3
