diff --git a/raster/test/regress/rt_utility_expected b/raster/test/regress/rt_utility_expected
index 2eb99e50b..7894cd0b2 100644
--- a/raster/test/regress/rt_utility_expected
+++ b/raster/test/regress/rt_utility_expected
@@ -1,23 +1,15 @@
 NOTICE:  Latitude and longitude required for computing pixel row and column of a rotated raster
-NOTICE:  Latitude and longitude required for computing pixel row and column of a rotated raster
 test 2.4|2|1217x1156, ip:782325.5,26744042.5 scale:5,-5 skew:3,3 srid:9102707 width:1217 height:1156
 test 2.4|4|6000x6000, ip:-75.5533328537098,49.2824585505576 scale:0.000805965234044584,-0.00080596523404458 skew:0.000211812383858707,0.000211812383858704 srid:4326 width:6000 height:6000
 ERROR:  Raster and geometry do not have the same SRID
-ERROR:  Raster and geometry do not have the same SRID
-ERROR:  Raster and geometry do not have the same SRID
-NOTICE:  Latitude and longitude required for computing pixel row and column of a rotated raster
 NOTICE:  Latitude and longitude required for computing pixel row and column of a rotated raster
 test 5.4|2|1217x1156, ip:782325.5,26744042.5 scale:5,-5 skew:3,3 srid:9102707 width:1217 height:1156
 test 5.4|4|6000x6000, ip:-75.5533328537098,49.2824585505576 scale:0.000805965234044584,-0.00080596523404458 skew:0.000211812383858707,0.000211812383858704 srid:4326 width:6000 height:6000
 ERROR:  Raster and geometry do not have the same SRID
-ERROR:  Raster and geometry do not have the same SRID
-ERROR:  Raster and geometry do not have the same SRID
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
 NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
 test 8.3|2|1217x1156, ip:782325.5,26744042.5 scale:5,-5 skew:3,3 srid:9102707 width:1217 height:1156
 test 8.3|4|6000x6000, ip:-75.5533328537098,49.2824585505576 scale:0.000805965234044584,-0.00080596523404458 skew:0.000211812383858707,0.000211812383858704 srid:4326 width:6000 height:6000
 NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
-NOTICE:  Pixel row and column required for computing longitude and latitude of a rotated raster
 test 10.3|2|1217x1156, ip:782325.5,26744042.5 scale:5,-5 skew:3,3 srid:9102707 width:1217 height:1156
 test 10.3|4|6000x6000, ip:-75.5533328537098,49.2824585505576 scale:0.000805965234044584,-0.00080596523404458 skew:0.000211812383858707,0.000211812383858704 srid:4326 width:6000 height:6000
 test 11.1|t
