diff --git a/extensions/postgis_topology/sql_bits/remove_from_extension.sql.in b/extensions/postgis_topology/sql_bits/remove_from_extension.sql.in
index 9d695d6a3..3ce17e448 100644
--- a/extensions/postgis_topology/sql_bits/remove_from_extension.sql.in
+++ b/extensions/postgis_topology/sql_bits/remove_from_extension.sql.in
@@ -5,6 +5,7 @@
 -- http://postgis.net
 --
 -- Copyright (C) 2011 Regina Obe <lr@pcorp.us>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 -- This is free software; you can redistribute and/or modify it under
 -- the terms of the GNU General Public Licence. See the COPYING file.
