diff --git a/loader/Makefile.in b/loader/Makefile.in
index f20670e3f..9e90128b6 100644
--- a/loader/Makefile.in
+++ b/loader/Makefile.in
@@ -3,6 +3,7 @@
 # * PostGIS - Spatial Types for PostgreSQL
 # * http://postgis.net
 # * Copyright 2008 Mark Cave-Ayland
+# * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 # *
 # * This is free software; you can redistribute and/or modify it under
 # * the terms of the GNU General Public Licence. See the COPYING file.
