diff --git a/loader/pgsql2shp-core.h b/loader/pgsql2shp-core.h
index 205714b98..093877084 100644
--- a/loader/pgsql2shp-core.h
+++ b/loader/pgsql2shp-core.h
@@ -5,6 +5,7 @@
  * Copyright 2001-2003 Refractions Research Inc.
  * Copyright 2009 Paul Ramsey <pramsey@cleverelephant.ca>
  * Copyright 2009 Mark Cave-Ayland <mark.cave-ayland@siriusit.co.uk>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  * This is free software; you can redistribute and/or modify it under
  * the terms of the GNU General Public Licence. See the COPYING file.
