diff --git a/loader/shp2pgsql-cli.c b/loader/shp2pgsql-cli.c
index 6271b7659..11380cd05 100644
--- a/loader/shp2pgsql-cli.c
+++ b/loader/shp2pgsql-cli.c
@@ -4,6 +4,7 @@
  * http://www.postgis.org
  * Copyright 2008 OpenGeo.org
  * Copyright 2009 Mark Cave-Ayland <mark.cave-ayland@siriusit.co.uk>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  * This is free software; you can redistribute and/or modify it under
  * the terms of the GNU General Public Licence. See the COPYING file.
