diff --git a/extensions/address_standardizer/gamma.c b/extensions/address_standardizer/gamma.c
index dcb16f7d0..32647809b 100644
--- a/extensions/address_standardizer/gamma.c
+++ b/extensions/address_standardizer/gamma.c
@@ -5,6 +5,7 @@ lookup structures. These are based on the optimized Aho-Corasick
 algorithms in Watson (1994).
 
 Copyright (c) 2008 Walter Bruce Sinclair
+Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 
 Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
 
