diff --git a/extensions/address_standardizer/hash.c b/extensions/address_standardizer/hash.c
index ba11a43f7..d291fe21a 100644
--- a/extensions/address_standardizer/hash.c
+++ b/extensions/address_standardizer/hash.c
@@ -3,6 +3,7 @@
 // hash.c
 //
 // Copyright (c) 2012 TJ Holowaychuk <tj@vision-media.ca>
+// Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 //
 
 #include "hash.h"
