diff --git a/extensions/address_standardizer/pagc_api.h b/extensions/address_standardizer/pagc_api.h
index 18e4ab3d4..1347d1fbc 100644
--- a/extensions/address_standardizer/pagc_api.h
+++ b/extensions/address_standardizer/pagc_api.h
@@ -5,6 +5,7 @@ This file is the common header file for PAGC internal routines
 Prototype 20H10 (This file was written by Walter Sinclair).
 
 Copyright (c) 2001-2012 Walter Bruce Sinclair
+Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 
 Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
 
