diff --git a/extensions/address_standardizer/pagc_common.h b/extensions/address_standardizer/pagc_common.h
index 684f9a422..1fa091366 100644
--- a/extensions/address_standardizer/pagc_common.h
+++ b/extensions/address_standardizer/pagc_common.h
@@ -7,6 +7,7 @@ Prototype 20H10 (This file was written by Walter Sinclair).
 This file is part of PAGC.
 
 Copyright (c) 2010 Walter Bruce Sinclair
+Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 
 Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
 
