diff --git a/extensions/address_standardizer/us_gaz.sql b/extensions/address_standardizer/us_gaz.sql
index 8ac22528d..6bc26780a 100644
--- a/extensions/address_standardizer/us_gaz.sql
+++ b/extensions/address_standardizer/us_gaz.sql
@@ -4,7 +4,7 @@ CREATE TABLE IF NOT EXISTS us_gaz (
     word text,
     stdword text,
     token integer, is_custom boolean NOT NULL DEFAULT true, CONSTRAINT pk_us_gaz PRIMARY KEY(id)
-);
+)DISTRIBUTED REPLICATED;
 
 -- this will be needed for upgrading to prevent purging of custom entries
 DELETE FROM us_gaz WHERE is_custom = false;
