diff --git a/topology/sql/topoelement/topoelement_agg.sql.in b/topology/sql/topoelement/topoelement_agg.sql.in
index af3a78fd0..e7355cc22 100644
--- a/topology/sql/topoelement/topoelement_agg.sql.in
+++ b/topology/sql/topoelement/topoelement_agg.sql.in
@@ -4,6 +4,7 @@
 -- http://postgis.net
 --
 -- Copyright (C) 2010, 2011 Sandro Santilli <strk@kbt.io>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 -- This is free software; you can redistribute and/or modify it under
 -- the terms of the GNU General Public Licence. See the COPYING file.
