diff --git a/utils/create_unpackaged.pl b/utils/create_unpackaged.pl
index 84f7aeeb9..4e641f22b 100755
--- a/utils/create_unpackaged.pl
+++ b/utils/create_unpackaged.pl
@@ -5,6 +5,7 @@
 # http://postgis.net
 #
 # Copyright (C) 2013 Sandro Santilli <strk@kbt.io>
+# Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 #
 # This is free software; you can redistribute and/or modify it under
 # the terms of the GNU General Public Licence. See the COPYING file.
