diff --git a/raster/test/regress/rt_dumpvalues.sql b/raster/test/regress/rt_dumpvalues.sql
index 2cf67f042..dfd074d0c 100644
--- a/raster/test/regress/rt_dumpvalues.sql
+++ b/raster/test/regress/rt_dumpvalues.sql
@@ -87,14 +87,14 @@ SELECT
 	rid,
 	(ST_DumpValues(rast)).*
 FROM raster_dumpvalues
-ORDER BY rid;
+ORDER BY 1,2;
 
 SELECT
 	rid,
 	(ST_DumpValues(rast, ARRAY[3,2,1])).*
 FROM raster_dumpvalues
 WHERE rid > 20
-ORDER BY rid;
+ORDER BY 1,2 DESC;
 
 DROP TABLE IF EXISTS raster_dumpvalues;
 
