diff --git a/raster/test/regress/rt_georeference_expected b/raster/test/regress/rt_georeference_expected
index 8c03f46a6..285d17d86 100644
--- a/raster/test/regress/rt_georeference_expected
+++ b/raster/test/regress/rt_georeference_expected
@@ -17,7 +17,6 @@
 5.0000000000EOL1.0000000000EOL1.0000000000EOL5.0000000000EOL10.0000000000EOL5.0000000000EOL|t
 5.0000000000EOL7.0000000000EOL3.0000000000EOL5.0000000000EOL10.0000000000EOL5.0000000000EOL|t
 ERROR:  st_setgeoreference requires a string with 6 floating point values.
-ERROR:  st_setgeoreference requires a string with 6 floating point values.
 t
 t
 t|t|t|t
