diff --git a/raster/test/regress/rt_gdalwarp_expected b/raster/test/regress/rt_gdalwarp_expected
index 531e407fe..dd00a4e04 100644
--- a/raster/test/regress/rt_gdalwarp_expected
+++ b/raster/test/regress/rt_gdalwarp_expected
@@ -1,12 +1,3 @@
-NOTICE:  table "raster_gdalwarp_src" does not exist, skipping
-NOTICE:  table "raster_gdalwarp_dst" does not exist, skipping
-NOTICE:  Values must be provided for both X and Y when specifying the scale.  Returning original raster
-NOTICE:  Values must be provided for both X and Y when specifying the scale.  Returning original raster
-NOTICE:  Raster has default geotransform. Adjusting metadata for use of GDAL Warp API
-NOTICE:  Raster has default geotransform. Adjusting metadata for use of GDAL Warp API
-NOTICE:  Raster has default geotransform. Adjusting metadata for use of GDAL Warp API
-NOTICE:  Values must be provided for both X and Y when specifying the scale.  Returning original raster
-NOTICE:  Values must be provided for both X and Y when specifying the scale.  Returning original raster
 0.0|||||||||||||
 0.1|993310|12|12|1|1009.894|-1009.894|0.000|0.000|950732.188|1409281.783|t|t|t
 0.10|992163|10|10|1|1000.000|-1000.000|0.000|0.000|-500000.000|600000.000|t|t|t
@@ -111,16 +102,9 @@ NOTICE:  Values must be provided for both X and Y when specifying the scale.  Re
 5.7|992163|10|11|1|1000.000|-1000.000|0.000|0.000|-500000.000|600991.000|t|t|t
 5.8|992163|10|11|1|1000.000|-1000.000|0.000|0.000|-500000.000|600001.000|t|t|t
 5.9|992163|10|11|1|1000.000|-1000.000|0.000|0.000|-500000.000|600009.000|t|t|t
-NOTICE:  The rasters (pixel corner coordinates) are not aligned
 t|f|t
-NOTICE:  Raster has default geotransform. Adjusting metadata for use of GDAL Warp API
-NOTICE:  Raster has default geotransform. Adjusting metadata for use of GDAL Warp API
-NOTICE:  Raster has default geotransform. Adjusting metadata for use of GDAL Warp API
-NOTICE:  Raster has default geotransform. Adjusting metadata for use of GDAL Warp API
 1|0|0|500|500|2|-2|0|0|0|1|250000|63750000|255|0|255|255
 2|0|0|500|100|2|-10|0|0|0|1|50000|12750000|255|0|255|255
 3|0|0|250|800|4|-1.25|0|0|0|1|200000|51000000|255|0|255|255
 4|0|0|500|500|2|-2|0|0|0|1|250000|63750000|255|0|255|255
-NOTICE:  Raster has default geotransform. Adjusting metadata for use of GDAL Warp API
-NOTICE:  Raster has default geotransform. Adjusting metadata for use of GDAL Warp API
 (0,0,5,5,2,-2,0,0,0,1)|(0,0,5,5,2,-2,0,0,0,1)
