diff --git a/raster/test/regress/rt_isempty.sql b/raster/test/regress/rt_isempty.sql
index 5304921bb..2c5894acb 100644
--- a/raster/test/regress/rt_isempty.sql
+++ b/raster/test/regress/rt_isempty.sql
@@ -1,6 +1,7 @@
 -----------------------------------------------------------------------
 --
 -- Copyright (c) 2011 Jorge Arevalo <jorge.arevalo@deimos-space.com>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 -- This program is free software; you can redistribute it and/or
 -- modify it under the terms of the GNU General Public License
