diff --git a/raster/test/regress/rt_mapalgebrafct_expected b/raster/test/regress/rt_mapalgebrafct_expected
index 75507297e..9868ab27e 100644
--- a/raster/test/regress/rt_mapalgebrafct_expected
+++ b/raster/test/regress/rt_mapalgebrafct_expected
@@ -1,6 +1,4 @@
-WARNING:  Raster is NULL. Returning NULL
 t
-WARNING:  Raster is NULL. Returning NULL
 t
 NOTICE:  Raster is empty. Returning an empty raster
 t
