diff --git a/raster/test/regress/rt_mapalgebrafct_2raster.sql b/raster/test/regress/rt_mapalgebrafct_2raster.sql
index 7e5fb9781..80080e5af 100644
--- a/raster/test/regress/rt_mapalgebrafct_2raster.sql
+++ b/raster/test/regress/rt_mapalgebrafct_2raster.sql
@@ -1,3 +1,4 @@
+SET client_min_messages TO WARNING;
 DROP TABLE IF EXISTS raster_mapalgebra;
 CREATE TABLE raster_mapalgebra (
 	rid integer,
@@ -333,7 +334,7 @@ FROM (
 		ST_Value(rast, 1, 1, 1) AS firstvalue,
 		ST_Value(rast, 1, ST_Width(rast), ST_Height(rast)) AS lastvalue
 	FROM raster_mapalgebra_out
-) AS r;
+) AS r ORDER BY 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16;
 
 DROP FUNCTION IF EXISTS raster_mapalgebra_intersection(double precision, double precision, int[], VARIADIC text[]);
 DROP FUNCTION IF EXISTS raster_mapalgebra_union(double precision, double precision, VARIADIC text[]);
@@ -342,3 +343,5 @@ DROP FUNCTION IF EXISTS raster_mapalgebra_second(double precision, double precis
 
 DROP TABLE IF EXISTS raster_mapalgebra;
 DROP TABLE IF EXISTS raster_mapalgebra_out;
+
+RESET client_min_messages;
