diff --git a/raster/test/regress/rt_worldtorastercoord.sql b/raster/test/regress/rt_worldtorastercoord.sql
index 38b1679aa..a228c56bd 100644
--- a/raster/test/regress/rt_worldtorastercoord.sql
+++ b/raster/test/regress/rt_worldtorastercoord.sql
@@ -48,137 +48,137 @@ DROP FUNCTION make_test_raster(integer, integer, integer, double precision, doub
 SELECT
 	rid,
 	(ST_WorldToRasterCoord(rast, -2, -2)).*
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	(ST_WorldToRasterCoord(rast, 0, 0)).*
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	(ST_WorldToRasterCoord(rast, 1, -1)).*
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	(ST_WorldToRasterCoord(rast, 1, 1)).*
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	(ST_WorldToRasterCoord(rast, 2, 2)).*
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 
 SELECT
 	rid,
 	(ST_WorldToRasterCoord(rast, ST_MakePoint(-2, -2))).*
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	(ST_WorldToRasterCoord(rast, ST_MakePoint(0, 0))).*
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	(ST_WorldToRasterCoord(rast, ST_MakePoint(1, -1))).*
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	(ST_WorldToRasterCoord(rast, ST_MakePoint(1, 1))).*
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	(ST_WorldToRasterCoord(rast, ST_MakePoint(2, 2))).*
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 
 SELECT
 	rid,
 	ST_WorldToRasterCoordX(rast, -2, -2)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	ST_WorldToRasterCoordX(rast, 0, 0)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	ST_WorldToRasterCoordX(rast, 1, -1)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	ST_WorldToRasterCoordX(rast, 1, 1)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	ST_WorldToRasterCoordX(rast, 2, 2)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 
 SELECT
 	rid,
 	ST_WorldToRasterCoordX(rast, ST_MakePoint(-2, -2))
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 
 SELECT
 	rid,
 	ST_WorldToRasterCoordX(rast, -2)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	ST_WorldToRasterCoordX(rast, 0)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	ST_WorldToRasterCoordX(rast, 1)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	ST_WorldToRasterCoordX(rast, 1)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	ST_WorldToRasterCoordX(rast, 2)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 
 SELECT
 	rid,
 	ST_WorldToRasterCoordY(rast, -2, -2)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	ST_WorldToRasterCoordY(rast, 0, 0)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	ST_WorldToRasterCoordY(rast, 1, -1)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	ST_WorldToRasterCoordY(rast, 1, 1)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	ST_WorldToRasterCoordY(rast, 2, 2)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 
 SELECT
 	rid,
 	ST_WorldToRasterCoordX(rast, ST_MakePoint(-2, -2))
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 
 SELECT
 	rid,
 	ST_WorldToRasterCoordY(rast, -2)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	ST_WorldToRasterCoordY(rast, 0)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	ST_WorldToRasterCoordY(rast, 1)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	ST_WorldToRasterCoordY(rast, 1)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 SELECT
 	rid,
 	ST_WorldToRasterCoordY(rast, 2)
-FROM raster_world2raster;
+FROM raster_world2raster  order by rid;
 
 DROP TABLE raster_world2raster;
