diff --git a/raster/loader/Makefile.in b/raster/loader/Makefile.in
index 21d9f128a..ff8a9a782 100644
--- a/raster/loader/Makefile.in
+++ b/raster/loader/Makefile.in
@@ -3,6 +3,7 @@
 # Copyright (c) 2011 Regents of the University of California
 #   <bkpark@ucdavis.edu>
 # Copyright (c) 2009 Sandro Santilli <strk@kbt.io>
+# Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 #
 # This program is free software; you can redistribute it and/or
 # modify it under the terms of the GNU General Public License
