diff --git a/postgis/geobuf.LICENSE b/postgis/geobuf.LICENSE
index 7d01fbc4e..9673eb169 100644
--- a/postgis/geobuf.LICENSE
+++ b/postgis/geobuf.LICENSE
@@ -1,6 +1,7 @@
 ISC License
 
 Copyright (c) 2015, Mapbox
+Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 
 Permission to use, copy, modify, and/or distribute this software for any
 purpose with or without fee is hereby granted, provided that the above
