diff --git a/postgis/geography_inout.c b/postgis/geography_inout.c
index 298b6df17..407923bd1 100644
--- a/postgis/geography_inout.c
+++ b/postgis/geography_inout.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2009-2011 Paul Ramsey <pramsey@cleverelephant.ca>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
