diff --git a/postgis/geography_measurement_trees.c b/postgis/geography_measurement_trees.c
index d73b2c2b0..04e56f013 100644
--- a/postgis/geography_measurement_trees.c
+++ b/postgis/geography_measurement_trees.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * ^copyright^
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
