diff --git a/postgis/geography.sql.in b/postgis/geography.sql.in
index 1122dab66..21b2188cc 100644
--- a/postgis/geography.sql.in
+++ b/postgis/geography.sql.in
@@ -2,6 +2,7 @@
 --
 -- PostGIS - Spatial Types for PostgreSQL
 -- Copyright 2009 Paul Ramsey <pramsey@cleverelephant.ca>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 -- This is free software; you can redistribute and/or modify it under
 -- the terms of the GNU General Public Licence. See the COPYING file.
