diff --git a/postgis/lwgeom_functions_temporal.c b/postgis/lwgeom_functions_temporal.c
index 7eba96b59..342805a3a 100644
--- a/postgis/lwgeom_functions_temporal.c
+++ b/postgis/lwgeom_functions_temporal.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright (C) 2015 Sandro Santilli <strk@kbt.io>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
