diff --git a/postgis/lwgeom_box.c b/postgis/lwgeom_box.c
index 3c0b99ed2..7db96ddef 100644
--- a/postgis/lwgeom_box.c
+++ b/postgis/lwgeom_box.c
@@ -20,6 +20,7 @@
  *
  * Copyright 2001-2009 Refractions Research Inc.
  * Copyright 2009 Mark Cave-Ayland <mark.cave-ayland@siriusit.co.uk>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
