diff --git a/postgis/lwgeom_rectree.c b/postgis/lwgeom_rectree.c
index 17a774a42..f18c3c849 100644
--- a/postgis/lwgeom_rectree.c
+++ b/postgis/lwgeom_rectree.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright (C) 2018 Paul Ramsey
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
