diff --git a/postgis/lwgeom_geos_relatematch.c b/postgis/lwgeom_geos_relatematch.c
index d77141349..d03b15416 100644
--- a/postgis/lwgeom_geos_relatematch.c
+++ b/postgis/lwgeom_geos_relatematch.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright (C) 2010 Sandro Santilli <strk@kbt.io>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
