diff --git a/postgis/lwgeom_geos_clean.c b/postgis/lwgeom_geos_clean.c
index 12ce7797f..a2940a721 100644
--- a/postgis/lwgeom_geos_clean.c
+++ b/postgis/lwgeom_geos_clean.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * Copyright 2009-2010 Sandro Santilli <strk@kbt.io>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
