diff --git a/postgis/lwgeom_btree.c b/postgis/lwgeom_btree.c
index 95776a579..15427bdf6 100644
--- a/postgis/lwgeom_btree.c
+++ b/postgis/lwgeom_btree.c
@@ -21,6 +21,7 @@
  * Copyright (C) 2010 Olivier Courtin <olivier.courtin@oslandia.com>
  * Copyright (C) 2010 Mark Cave-Ayland <mark.cave-ayland@siriusit.co.uk>
  * Copyright (C) 2009-2011 Paul Ramsey <pramsey@cleverelephant.ca>
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
