diff --git a/postgis/lwgeom_inout.c b/postgis/lwgeom_inout.c
index c7becc19b..451a8a262 100644
--- a/postgis/lwgeom_inout.c
+++ b/postgis/lwgeom_inout.c
@@ -19,6 +19,7 @@
  **********************************************************************
  *
  * ^copyright^
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  *
  **********************************************************************/
 
