diff --git a/postgis/postgis_drop_after.sql b/postgis/postgis_drop_after.sql
index 9dabf0016..eef6a70e9 100644
--- a/postgis/postgis_drop_after.sql
+++ b/postgis/postgis_drop_after.sql
@@ -7,6 +7,7 @@
 -- Copyright (C) 2011-2012 Sandro Santilli <strk@kbt.io>
 -- Copyright (C) 2010-2012 Regina Obe <lr@pcorp.us>
 -- Copyright (C) 2009      Paul Ramsey <pramsey@cleverelephant.ca>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 -- This is free software; you can redistribute and/or modify it under
 -- the terms of the GNU General Public Licence. See the COPYING file.
@@ -145,6 +146,7 @@ DROP FUNCTION IF EXISTS st_geometry_eq(geometry, geometry);
 DROP FUNCTION IF EXISTS st_geometry_cmp(geometry, geometry);
 DROP FUNCTION IF EXISTS SnapToGrid(geometry, float8, float8);
 DROP FUNCTION IF EXISTS st_removerepeatedpoints(geometry);
+-- FIX_ME_POSTGIS_254: Re-enable, when adding support for GEOS 3.5
 DROP FUNCTION IF EXISTS st_voronoi(geometry, geometry, double precision, boolean); --temporarely introduced before 2.3.0 final
 
 DROP FUNCTION IF EXISTS geometry_gist_sel_2d (internal, oid, internal, int4);
@@ -175,11 +177,6 @@ DROP FUNCTION IF EXISTS _ST_DumpPoints( geometry, integer[]); -- removed 2.4.0,
 DROP FUNCTION IF EXISTS _ST_DistanceRectTree(g1 geometry, g2 geometry);
 DROP FUNCTION IF EXISTS _ST_DistanceRectTreeCached(g1 geometry, g2 geometry);
 
-
--- pgis_abs type was increased from 8 bytes in 2.1 to 16 bytes in 2.2
--- See #3460
-UPDATE pg_type SET typlen=16 WHERE typname='pgis_abs' AND typlen=8;
-
 DO language 'plpgsql'
 $$
 BEGIN
