diff --git a/postgis/postgis_drop_before.sql b/postgis/postgis_drop_before.sql
index 15eb7b261..fc3be8dfb 100644
--- a/postgis/postgis_drop_before.sql
+++ b/postgis/postgis_drop_before.sql
@@ -7,6 +7,7 @@
 -- Copyright (C) 2011-2012 Sandro Santilli <strk@kbt.io>
 -- Copyright (C) 2010-2013 Regina Obe <lr@pcorp.us>
 -- Copyright (C) 2009      Paul Ramsey <pramsey@cleverelephant.ca>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 -- This is free software; you can redistribute and/or modify it under
 -- the terms of the GNU General Public Licence. See the COPYING file.
