diff --git a/postgis/uninstall_long_xact.sql.in b/postgis/uninstall_long_xact.sql.in
index cda20e785..f1ab52cf4 100644
--- a/postgis/uninstall_long_xact.sql.in
+++ b/postgis/uninstall_long_xact.sql.in
@@ -4,6 +4,7 @@
 -- PostGIS - Spatial Types for PostgreSQL
 -- http://postgis.net
 -- Copyright 2001-2003 Refractions Research Inc.
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 -- This is free software; you can redistribute and/or modify it under
 -- the terms of the GNU General Public Licence. See the COPYING file.
