diff --git a/postgis/uninstall_geography.sql.in b/postgis/uninstall_geography.sql.in
index 60b3885f8..7cd11592b 100644
--- a/postgis/uninstall_geography.sql.in
+++ b/postgis/uninstall_geography.sql.in
@@ -4,6 +4,7 @@
 -- PostGIS - Spatial Types for PostgreSQL
 -- http://postgis.org/
 -- Copyright 2009 Paul Ramsey <pramsey@cleverelephant.ca>
+-- Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 --
 -- This is free software; you can redistribute and/or modify it under
 -- the terms of the GNU General Public Licence. See the COPYING file.
