diff --git a/postgis/uthash.h b/postgis/uthash.h
index 5758e9fe7..2d2328f6e 100644
--- a/postgis/uthash.h
+++ b/postgis/uthash.h
@@ -1,6 +1,7 @@
 /*
 Copyright (c) 2003-2017, Troy D. Hanson     http://troydhanson.github.com/uthash/
 All rights reserved.
+Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 
 Redistribution and use in source and binary forms, with or without
 modification, are permitted provided that the following conditions are met:
