diff --git a/loader/cunit/Makefile.in b/loader/cunit/Makefile.in
index 8bc51fee2..29c0b8c8a 100644
--- a/loader/cunit/Makefile.in
+++ b/loader/cunit/Makefile.in
@@ -3,6 +3,7 @@
 # * PostGIS - Spatial Types for PostgreSQL
 # * http://postgis.net
 # * Copyright 2008 Paul Ramsey, Mark Cave-Ayland
+# * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 # *
 # * This is free software; you can redistribute and/or modify it under
 # * the terms of the GNU General Public Licence. See the COPYING file.
