diff --git a/macros/nls.m4 b/macros/nls.m4
index 7967cc2f9..111cef6b3 100644
--- a/macros/nls.m4
+++ b/macros/nls.m4
@@ -1,5 +1,6 @@
 # nls.m4 serial 3 (gettext-0.15)
 dnl Copyright (C) 1995-2003, 2005-2006 Free Software Foundation, Inc.
+dnl Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 dnl This file is free software; the Free Software Foundation
 dnl gives unlimited permission to copy and/or distribute it,
 dnl with or without modifications, as long as this notice is preserved.
