diff --git a/macros/progtest.m4 b/macros/progtest.m4
index a56365cd3..84c89448b 100644
--- a/macros/progtest.m4
+++ b/macros/progtest.m4
@@ -1,5 +1,6 @@
 # progtest.m4 serial 4 (gettext-0.14.2)
 dnl Copyright (C) 1996-2003, 2005 Free Software Foundation, Inc.
+dnl Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 dnl This file is free software; the Free Software Foundation
 dnl gives unlimited permission to copy and/or distribute it,
 dnl with or without modifications, as long as this notice is preserved.
