diff --git a/extras/tiger_geocoder/geocode/census_tracts_functions.sql b/extras/tiger_geocoder/geocode/census_tracts_functions.sql
index 564895710..ef344c51d 100644
--- a/extras/tiger_geocoder/geocode/census_tracts_functions.sql
+++ b/extras/tiger_geocoder/geocode/census_tracts_functions.sql
@@ -1,6 +1,7 @@
  /***
  *
  * Copyright (C) 2012 Regina Obe and Leo Hsu (Paragon Corporation)
+ * Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
  **/
 -- This function given a geometry try will try to determine the tract.
 -- It defaults to returning the tract name but can be changed to return track geoid id.
