diff --git a/doc/xsl/postgis_reference.xml.xsl b/doc/xsl/postgis_reference.xml.xsl
index 50d5b2e0a..eab3f17d0 100644
--- a/doc/xsl/postgis_reference.xml.xsl
+++ b/doc/xsl/postgis_reference.xml.xsl
@@ -2,6 +2,7 @@
 <!-- ********************************************************************
 	 ********************************************************************
 	 Copyright 2010, Regina Obe
+	 Modifications Copyright (c) 2017 - Present Pivotal Software, Inc. All Rights Reserved.
 	 License: BSD
 	 Purpose: This is an xsl transform that concatenates 
 	      all the sections defined in postgis.xml and strips all tags
